// @shopify/polaris-tokens/dist/esm/build/index.mjs
export { createMetadata, metadata } from '../src/metadata.mjs';
export { createExact, createVar, createVarName, getCustomPropertyNames, getKeyframeNames, getMediaConditions, getUnit, isKeyOf, rem, removeMetadata, toEm, toPx, toRem, tokensToRems } from '../src/utilities.mjs';
export { breakpointsAliases } from '../src/themes/base/breakpoints.mjs';
export { themeNameDefault } from '../src/themes/constants.mjs';

var border = {
  "border-radius-0": "0rem",
  "border-radius-050": "0.125rem",
  "border-radius-100": "0.25rem",
  "border-radius-150": "0.375rem",
  "border-radius-200": "0.5rem",
  "border-radius-300": "0.75rem",
  "border-radius-400": "1rem",
  "border-radius-500": "1.25rem",
  "border-radius-750": "1.875rem",
  "border-radius-full": "624.9375rem",
  "border-radius-0-experimental": "0rem",
  "border-radius-05": "0.125rem",
  "border-radius-1": "0.25rem",
  "border-radius-2": "0.5rem",
  "border-radius-3": "0.75rem",
  "border-radius-4": "1rem",
  "border-radius-5": "1.25rem",
  "border-radius-6": "1.875rem",
  "border-radius-1_5-experimental": "0.375rem",
  "border-width-0165": "0.04125rem",
  "border-width-025": "0.0625rem",
  "border-width-050": "0.125rem",
  "border-width-100": "0.25rem",
  "border-width-1": "0.0625rem",
  "border-width-2": "0.125rem",
  "border-width-3": "0.1875rem",
  "border-width-4": "0.25rem",
  "border-width-5": "0.3125rem",
  "border-width-1-experimental": "0.04125rem",
  "border-width-2-experimental": "0.0625rem"
};
var breakpoints = {
  "breakpoints-xs": "0rem",
  "breakpoints-sm": "30.625rem",
  "breakpoints-md": "48rem",
  "breakpoints-lg": "65rem",
  "breakpoints-xl": "90rem"
};
var color = {
  "color-border-inverse-active": "rgba(227, 227, 227, 1)",
  "color-border-inverse-hover": "rgba(204, 204, 204, 1)",
  "color-border-warning": "rgba(255, 200, 121, 1)",
  "color-checkbox-icon-disabled": "rgba(255, 255, 255, 1)",
  "color-checkbox-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
  "color-bg-fill-active": "rgba(247, 247, 247, 1)",
  "color-bg-fill-brand-selected": "rgba(48, 48, 48, 1)",
  "color-bg-fill-caution-active": "rgba(225, 203, 0, 1)",
  "color-bg-fill-caution-hover": "rgba(234, 211, 0, 1)",
  "color-bg-fill-critical-selected": "rgba(142, 31, 11, 1)",
  "color-bg-fill-emphasis-active": "rgba(0, 46, 106, 1)",
  "color-bg-fill-emphasis-hover": "rgba(0, 66, 153, 1)",
  "color-bg-fill-emphasis": "rgba(0, 91, 211, 1)",
  "color-bg-fill-hover": "rgba(250, 250, 250, 1)",
  "color-bg-fill-info-active": "rgba(0, 148, 213, 1)",
  "color-bg-fill-info-hover": "rgba(81, 192, 255, 1)",
  "color-bg-fill-secondary-active": "rgba(227, 227, 227, 1)",
  "color-bg-fill-secondary-hover": "rgba(235, 235, 235, 1)",
  "color-bg-fill-selected": "rgba(204, 204, 204, 1)",
  "color-bg-fill-transparent-secondary-active": "rgba(0, 0, 0, 0.11)",
  "color-bg-fill-transparent-secondary-hover": "rgba(0, 0, 0, 0.08)",
  "color-bg-fill-transparent-selected": "rgba(0, 0, 0, 0.06)",
  "color-bg-fill-transparent": "rgba(0, 0, 0, 0)",
  "color-bg-fill-warning-active": "rgba(178, 132, 0, 1)",
  "color-bg-fill-warning-hover": "rgba(229, 165, 0, 1)",
  "color-bg-fill": "rgba(255, 255, 255, 1)",
  "color-icon-secondary-active": "rgba(74, 74, 74, 1)",
  "color-icon-secondary-hover": "rgba(97, 97, 97, 1)",
  "color-text-link-active": "rgba(0, 46, 106, 1)",
  "color-text-link-hover": "rgba(0, 66, 153, 1)",
  "color-text-link": "rgba(0, 91, 211, 1)",
  "color-nav-bg": "rgba(235, 235, 235, 1)",
  "color-nav-bg-surface-active": "rgba(250, 250, 250, 1)",
  "color-nav-bg-surface-hover": "rgba(241, 241, 241, 1)",
  "color-nav-bg-surface-selected": "rgba(250, 250, 250, 1)",
  "color-nav-bg-surface": "rgba(0, 0, 0, 0.02)",
  "color-radio-button-icon-disabled": "rgba(255, 255, 255, 1)",
  "color-radio-button-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
  "color-bg-surface-emphasis-active": "rgba(226, 231, 255, 1)",
  "color-bg-surface-emphasis-hover": "rgba(234, 237, 255, 1)",
  "color-bg-surface-emphasis": "rgba(240, 242, 255, 1)",
  "color-bg-surface-inverse": "rgba(48, 48, 48, 1)",
  "color-bg-surface-magic-active": "rgba(228, 222, 255, 1)",
  "color-bg-surface-secondary-selected": "rgba(235, 235, 235, 1)",
  "color-bg-surface-tertiary-active": "rgba(227, 227, 227, 1)",
  "color-bg-surface-tertiary-hover": "rgba(235, 235, 235, 1)",
  "color-bg-surface-warning-active": "rgba(255, 228, 198, 1)",
  "color-bg-surface-warning-hover": "rgba(255, 235, 213, 1)",
  "color-text-brand-on-bg-fill-active": "rgba(204, 204, 204, 1)",
  "color-text-brand-on-bg-fill-disabled": "rgba(255, 255, 255, 1)",
  "color-text-brand-on-bg-fill-hover": "rgba(227, 227, 227, 1)",
  "color-text-brand-on-bg-fill": "rgba(255, 255, 255, 1)",
  "color-text-caution-active": "rgba(31, 28, 0, 1)",
  "color-text-caution-hover": "rgba(51, 46, 0, 1)",
  "color-text-caution-on-bg-fill": "rgba(51, 46, 0, 1)",
  "color-text-critical-on-bg-fill": "rgba(255, 251, 251, 1)",
  "color-text-emphasis-on-bg-fill-active": "rgba(213, 220, 255, 1)",
  "color-text-emphasis-on-bg-fill-hover": "rgba(226, 231, 255, 1)",
  "color-text-emphasis-on-bg-fill": "rgba(252, 253, 255, 1)",
  "color-text-info-active": "rgba(0, 33, 51, 1)",
  "color-text-info-hover": "rgba(0, 58, 90, 1)",
  "color-text-magic-on-bg-fill": "rgba(253, 253, 255, 1)",
  "color-text-success-active": "rgba(9, 42, 27, 1)",
  "color-text-success-hover": "rgba(8, 61, 37, 1)",
  "color-text-success-on-bg-fill": "rgba(248, 255, 251, 1)",
  "color-text-warning-active": "rgba(37, 26, 0, 1)",
  "color-text-warning-hover": "rgba(65, 45, 0, 1)",
  "color-text-warning-on-bg-fill": "rgba(37, 26, 0, 1)",
  "color-video-thumbnail-play-button-bg-fill-hover": "rgba(0, 0, 0, 0.81)",
  "color-video-thumbnail-play-button-bg-fill": "rgba(0, 0, 0, 0.71)",
  "color-video-thumbnail-play-button-text-on-bg-fill": "rgba(255, 255, 255, 1)",
  "color-bg-surface": "rgba(255, 255, 255, 1)",
  "color-bg-surface-hover": "rgba(247, 247, 247, 1)",
  "color-bg-surface-active": "rgba(243, 243, 243, 1)",
  "color-bg-surface-disabled": "rgba(0, 0, 0, 0.05)",
  "color-bg-surface-secondary": "rgba(249, 250, 251, 1)",
  "color-bg-surface-secondary-hover": "rgba(241, 242, 244, 1)",
  "color-bg-surface-secondary-active": "rgba(235, 236, 239, 1)",
  "color-bg-surface-tertiary": "rgba(243, 243, 243, 1)",
  "color-bg-fill-tertiary": "rgba(221, 224, 228, 1)",
  "color-bg-fill-tertiary-hover": "rgba(202, 206, 211, 1)",
  "color-bg-fill-tertiary-active": "rgba(202, 206, 211, 1)",
  "color-input-bg-surface": "rgba(255, 255, 255, 1)",
  "color-input-bg-surface-hover": "rgba(250, 250, 250, 1)",
  "color-input-bg-surface-active": "rgba(247, 247, 247, 1)",
  "color-bg-fill-brand": "rgba(48, 48, 48, 1)",
  "color-bg-fill-brand-hover": "rgba(26, 26, 26, 1)",
  "color-bg-fill-brand-active": "rgba(26, 26, 26, 1)",
  "color-bg-surface-brand": "rgba(227, 227, 227, 1)",
  "color-bg-surface-brand-hover": "rgba(235, 235, 235, 1)",
  "color-bg-surface-brand-active": "rgba(241, 241, 241, 1)",
  "color-bg-surface-brand-selected": "rgba(241, 241, 241, 1)",
  "color-bg-surface-selected": "rgba(235, 236, 239, 1)",
  "color-bg-fill-success": "rgba(22, 166, 121, 1)",
  "color-bg-fill-success-hover": "rgba(19, 111, 69, 1)",
  "color-bg-fill-success-active": "rgba(12, 81, 50, 1)",
  "color-bg-fill-success-secondary": "rgba(161, 237, 208, 1)",
  "color-bg-surface-success": "rgba(224, 248, 238, 1)",
  "color-bg-surface-success-hover": "rgba(240, 253, 248, 1)",
  "color-bg-surface-success-active": "rgba(192, 242, 221, 1)",
  "color-bg-fill-critical": "rgba(197, 40, 12, 1)",
  "color-bg-fill-critical-hover": "rgba(159, 32, 10, 1)",
  "color-bg-fill-critical-active": "rgba(115, 24, 7, 1)",
  "color-bg-fill-critical-secondary": "rgba(251, 197, 188, 1)",
  "color-bg-surface-critical": "rgba(253, 226, 221, 1)",
  "color-bg-surface-critical-hover": "rgba(254, 243, 241, 1)",
  "color-bg-surface-critical-active": "rgba(251, 197, 188, 1)",
  "color-bg-fill-caution": "rgba(216, 155, 13, 1)",
  "color-bg-fill-caution-secondary": "rgba(248, 217, 144, 1)",
  "color-bg-surface-caution": "rgba(252, 240, 212, 1)",
  "color-bg-surface-caution-hover": "rgba(254, 248, 236, 1)",
  "color-bg-surface-caution-active": "rgba(250, 229, 178, 1)",
  "color-bg-fill-info": "rgba(42, 172, 187, 1)",
  "color-bg-fill-info-secondary": "rgba(184, 233, 239, 1)",
  "color-bg-surface-info": "rgba(222, 245, 247, 1)",
  "color-bg-surface-info-hover": "rgba(238, 250, 251, 1)",
  "color-bg-surface-info-active": "rgba(184, 233, 239, 1)",
  "color-bg-fill-warning": "rgba(255, 184, 0, 1)",
  "color-bg-fill-warning-secondary": "rgba(250, 201, 168, 1)",
  "color-bg-surface-warning": "rgba(255, 241, 227, 1)",
  "color-bg-fill-magic": "rgba(121, 69, 227, 1)",
  "color-bg-fill-magic-secondary": "rgba(236, 227, 253, 1)",
  "color-bg-fill-magic-secondary-hover": "rgba(226, 214, 250, 1)",
  "color-bg-fill-magic-secondary-active": "rgba(203, 180, 248, 1)",
  "color-bg-surface-magic": "rgba(243, 241, 255, 1)",
  "color-bg-surface-magic-hover": "rgba(236, 227, 253, 1)",
  "color-bg-fill-secondary": "rgba(241, 242, 244, 1)",
  "color-bg-fill-inverse": "rgba(97, 106, 117, 1)",
  "color-bg-fill-inverse-hover": "rgba(97, 106, 117, 1)",
  "color-bg-fill-inverse-active": "rgba(135, 144, 155, 1)",
  "color-bg-surface-transparent": "rgba(0, 0, 0, 0)",
  "color-bg-fill-transparent-hover": "rgba(0, 0, 0, 0.05)",
  "color-bg-fill-transparent-active": "rgba(0, 0, 0, 0.06)",
  "color-bg-fill-disabled": "rgba(0, 0, 0, 0.05)",
  "color-bg-fill-transparent-secondary": "rgba(0, 0, 0, 0.06)",
  "color-bg-fill-brand-disabled": "rgba(0, 0, 0, 0.17)",
  "color-backdrop-bg": "rgba(0, 0, 0, 0.71)",
  "color-avatar-bg-fill": "rgba(181, 181, 181, 1)",
  "color-avatar-one-bg-fill": "rgba(251, 207, 251, 1)",
  "color-avatar-two-bg-fill": "rgba(56, 250, 163, 1)",
  "color-avatar-three-bg-fill": "rgba(146, 237, 222, 1)",
  "color-avatar-four-bg-fill": "rgba(192, 225, 255, 1)",
  "color-avatar-five-bg-fill": "rgba(254, 209, 221, 1)",
  "color-text-secondary": "rgba(97, 106, 117, 1)",
  "color-text-emphasis": "rgba(36, 99, 188, 1)",
  "color-text-emphasis-hover": "rgba(20, 73, 149, 1)",
  "color-text-emphasis-active": "rgba(14, 53, 108, 1)",
  "color-text-brand": "rgba(0, 122, 92, 1)",
  "color-text-brand-hover": "rgba(18, 84, 67, 1)",
  "color-text-critical-hover": "rgba(95, 21, 7, 1)",
  "color-text-info-on-bg-fill": "rgba(16, 65, 71, 1)",
  "color-text-warning": "rgba(94, 66, 0, 1)",
  "color-text-inverse-secondary": "rgba(171, 177, 186, 1)",
  "color-text-link-inverse": "rgba(197, 208, 255, 1)",
  "color-avatar-text-on-bg-fill": "rgba(255, 255, 255, 1)",
  "color-avatar-one-text-on-bg-fill": "rgba(121, 26, 121, 1)",
  "color-avatar-two-text-on-bg-fill": "rgba(12, 81, 50, 1)",
  "color-avatar-three-text-on-bg-fill": "rgba(9, 83, 70, 1)",
  "color-avatar-four-text-on-bg-fill": "rgba(0, 82, 124, 1)",
  "color-avatar-five-text-on-bg-fill": "rgba(141, 4, 72, 1)",
  "color-icon-secondary": "rgba(135, 144, 155, 1)",
  "color-icon-emphasis": "rgba(36, 99, 188, 1)",
  "color-icon-emphasis-hover": "rgba(20, 73, 149, 1)",
  "color-icon-emphasis-active": "rgba(14, 53, 108, 1)",
  "color-icon-brand": "rgba(0, 122, 92, 1)",
  "color-border-secondary": "rgba(221, 224, 228, 1)",
  "color-border-tertiary": "rgba(204, 204, 204, 1)",
  "color-input-border": "rgba(171, 177, 186, 1)",
  "color-input-border-hover": "rgba(97, 106, 117, 1)",
  "color-input-border-active": "rgba(26, 26, 26, 1)",
  "color-border-emphasis": "rgba(0, 91, 211, 1)",
  "color-border-emphasis-hover": "rgba(20, 73, 149, 1)",
  "color-border-emphasis-active": "rgba(14, 53, 108, 1)",
  "color-border-focus": "rgba(62, 125, 213, 1)",
  "color-border-brand": "rgba(0, 122, 92, 1)",
  "color-border-critical-secondary": "rgba(142, 31, 11, 1)",
  "color-border-magic-secondary": "rgba(121, 69, 227, 1)",
  "color-bg-inverse": "rgba(31, 33, 36, 1)",
  "color-bg-inset-strong": "rgba(97, 106, 117, 1)",
  "color-bg-inverse-hover": "rgba(97, 106, 117, 1)",
  "color-bg-inverse-active": "rgba(135, 144, 155, 1)",
  "color-bg-strong-hover": "rgba(202, 206, 211, 1)",
  "color-bg-strong-active": "rgba(202, 206, 211, 1)",
  "color-bg-strong": "rgba(221, 224, 228, 1)",
  "color-bg-subdued-active": "rgba(235, 236, 239, 1)",
  "color-bg-disabled": "rgba(235, 236, 239, 1)",
  "color-bg-interactive-disabled": "rgba(235, 236, 239, 1)",
  "color-bg-app": "rgba(241, 242, 244, 1)",
  "color-bg-app-active": "rgba(221, 224, 228, 1)",
  "color-bg-app-hover": "rgba(235, 236, 239, 1)",
  "color-bg-app-selected": "rgba(235, 236, 239, 1)",
  "color-bg-active": "rgba(235, 236, 239, 1)",
  "color-bg-subdued-hover": "rgba(241, 242, 244, 1)",
  "color-bg-inset": "rgba(241, 242, 244, 1)",
  "color-bg-hover": "rgba(241, 242, 244, 1)",
  "color-bg-subdued": "rgba(249, 250, 251, 1)",
  "color-bg-input": "rgba(255, 255, 255, 1)",
  "color-bg": "rgba(255, 255, 255, 1)",
  "color-bg-primary-active": "rgba(12, 59, 47, 1)",
  "color-bg-primary-hover": "rgba(18, 84, 67, 1)",
  "color-bg-primary": "rgba(0, 122, 92, 1)",
  "color-bg-success-strong": "rgba(22, 166, 121, 1)",
  "color-bg-success": "rgba(161, 237, 208, 1)",
  "color-bg-primary-subdued-active": "rgba(192, 242, 221, 1)",
  "color-bg-success-subdued-active": "rgba(192, 242, 221, 1)",
  "color-bg-success-subdued": "rgba(224, 248, 238, 1)",
  "color-bg-primary-subdued-hover": "rgba(224, 248, 238, 1)",
  "color-bg-success-subdued-hover": "rgba(240, 253, 248, 1)",
  "color-bg-primary-subdued": "rgba(240, 253, 248, 1)",
  "color-bg-primary-subdued-selected": "rgba(240, 253, 248, 1)",
  "color-bg-critical-strong-active": "rgba(115, 24, 7, 1)",
  "color-bg-critical-strong-hover": "rgba(159, 32, 10, 1)",
  "color-bg-critical-strong": "rgba(197, 40, 12, 1)",
  "color-bg-critical-subdued-active": "rgba(251, 197, 188, 1)",
  "color-bg-critical": "rgba(251, 197, 188, 1)",
  "color-bg-critical-subdued": "rgba(253, 226, 221, 1)",
  "color-bg-critical-subdued-hover": "rgba(254, 243, 241, 1)",
  "color-bg-caution-strong": "rgba(216, 155, 13, 1)",
  "color-bg-caution": "rgba(248, 217, 144, 1)",
  "color-bg-caution-subdued-active": "rgba(250, 229, 178, 1)",
  "color-bg-caution-subdued": "rgba(252, 240, 212, 1)",
  "color-bg-caution-subdued-hover": "rgba(254, 248, 236, 1)",
  "color-bg-info-strong": "rgba(42, 172, 187, 1)",
  "color-bg-info-subdued-active": "rgba(184, 233, 239, 1)",
  "color-bg-info": "rgba(184, 233, 239, 1)",
  "color-bg-info-subdued": "rgba(222, 245, 247, 1)",
  "color-bg-info-subdued-hover": "rgba(238, 250, 251, 1)",
  "color-bg-interactive-active": "rgba(14, 53, 108, 1)",
  "color-bg-interactive-hover": "rgba(20, 73, 149, 1)",
  "color-bg-interactive": "rgba(36, 99, 188, 1)",
  "color-bg-interactive-subdued-active": "rgba(187, 212, 247, 1)",
  "color-bg-interactive-subdued-hover": "rgba(232, 240, 253, 1)",
  "color-bg-interactive-subdued": "rgba(240, 245, 253, 1)",
  "color-bg-interactive-selected": "rgba(240, 245, 253, 1)",
  "color-bg-warning": "rgba(250, 201, 168, 1)",
  "color-bg-magic-strong": "rgba(121, 69, 227, 1)",
  "color-bg-magic-hover": "rgba(226, 214, 250, 1)",
  "color-bg-magic-active": "rgba(203, 180, 248, 1)",
  "color-bg-magic": "rgba(236, 227, 253, 1)",
  "color-bg-magic-subdued-hover": "rgba(236, 227, 253, 1)",
  "color-bg-magic-subdued-active": "rgba(226, 214, 250, 1)",
  "color-bg-magic-subdued": "rgba(242, 237, 253, 1)",
  "color-border-input-hover": "rgba(97, 106, 117, 1)",
  "color-border-inverse": "rgba(97, 106, 117, 1)",
  "color-border-strong-hover": "rgba(135, 144, 155, 1)",
  "color-border-input": "rgba(171, 177, 186, 1)",
  "color-border-hover": "rgba(171, 177, 186, 1)",
  "color-border-strong": "rgba(171, 177, 186, 1)",
  "color-border": "rgba(202, 206, 211, 1)",
  "color-border-disabled": "rgba(221, 224, 228, 1)",
  "color-border-subdued": "rgba(221, 224, 228, 1)",
  "color-border-interactive-disabled": "rgba(221, 224, 228, 1)",
  "color-border-primary": "rgba(0, 122, 92, 1)",
  "color-border-success": "rgba(22, 166, 121, 1)",
  "color-border-success-subdued": "rgba(80, 220, 169, 1)",
  "color-border-critical-active": "rgba(67, 14, 4, 1)",
  "color-border-critical-hover": "rgba(115, 24, 7, 1)",
  "color-border-critical": "rgba(197, 40, 12, 1)",
  "color-border-critical-subdued": "rgba(245, 107, 82, 1)",
  "color-border-caution": "rgba(216, 155, 13, 1)",
  "color-border-caution-subdued": "rgba(245, 196, 82, 1)",
  "color-border-info": "rgba(59, 195, 211, 1)",
  "color-border-info-subdued": "rgba(109, 211, 222, 1)",
  "color-border-interactive-active": "rgba(14, 53, 108, 1)",
  "color-border-interactive-hover": "rgba(20, 73, 149, 1)",
  "color-border-interactive": "rgba(62, 125, 213, 1)",
  "color-border-interactive-focus": "rgba(62, 125, 213, 1)",
  "color-border-interactive-subdued": "rgba(187, 212, 247, 1)",
  "color-border-magic-strong": "rgba(121, 69, 227, 1)",
  "color-border-magic": "rgba(173, 139, 241, 1)",
  "color-icon-hover": "rgba(31, 33, 36, 1)",
  "color-icon": "rgba(97, 106, 117, 1)",
  "color-icon-active": "rgba(31, 33, 36, 1)",
  "color-icon-subdued": "rgba(135, 144, 155, 1)",
  "color-icon-disabled": "rgba(171, 177, 186, 1)",
  "color-icon-interactive-disabled": "rgba(171, 177, 186, 1)",
  "color-icon-inverse": "rgba(221, 224, 228, 1)",
  "color-icon-on-color": "rgba(255, 255, 255, 1)",
  "color-icon-primary": "rgba(0, 122, 92, 1)",
  "color-icon-success": "rgba(22, 166, 121, 1)",
  "color-icon-critical": "rgba(197, 40, 12, 1)",
  "color-icon-caution": "rgba(183, 126, 11, 1)",
  "color-icon-info": "rgba(42, 172, 187, 1)",
  "color-icon-warning": "rgba(242, 117, 34, 1)",
  "color-icon-interactive-active": "rgba(14, 53, 108, 1)",
  "color-icon-interactive-hover": "rgba(20, 73, 149, 1)",
  "color-icon-interactive": "rgba(36, 99, 188, 1)",
  "color-icon-interactive-inverse": "rgba(102, 153, 225, 1)",
  "color-icon-magic": "rgba(121, 69, 227, 1)",
  "color-text": "rgba(31, 33, 36, 1)",
  "color-text-subdued": "rgba(97, 106, 117, 1)",
  "color-text-disabled": "rgba(135, 144, 155, 1)",
  "color-text-interactive-disabled": "rgba(135, 144, 155, 1)",
  "color-text-inverse-subdued": "rgba(171, 177, 186, 1)",
  "color-text-inverse": "rgba(241, 242, 244, 1)",
  "color-text-on-color": "rgba(255, 255, 255, 1)",
  "color-text-success-strong": "rgba(12, 59, 47, 1)",
  "color-text-success": "rgba(0, 122, 92, 1)",
  "color-text-primary": "rgba(0, 122, 92, 1)",
  "color-text-primary-hover": "rgba(18, 84, 67, 1)",
  "color-text-critical-strong": "rgba(67, 14, 4, 1)",
  "color-text-critical-active": "rgba(115, 24, 7, 1)",
  "color-text-critical": "rgba(197, 40, 12, 1)",
  "color-text-caution-strong": "rgba(77, 46, 5, 1)",
  "color-text-caution": "rgba(135, 92, 8, 1)",
  "color-text-info-strong": "rgba(16, 65, 71, 1)",
  "color-text-info": "rgba(32, 130, 141, 1)",
  "color-text-warning-strong": "rgba(77, 36, 5, 1)",
  "color-text-interactive-active": "rgba(14, 53, 108, 1)",
  "color-text-interactive-hover": "rgba(20, 73, 149, 1)",
  "color-text-interactive": "rgba(36, 99, 188, 1)",
  "color-text-interactive-inverse": "rgba(102, 153, 225, 1)",
  "color-text-magic-strong": "rgba(49, 13, 120, 1)",
  "color-text-magic": "rgba(90, 36, 205, 1)",
  "color-bg-backdrop-experimental": "rgba(0, 0, 0, 0.71)",
  "color-bg-primary-disabled-experimental": "rgba(212, 212, 212, 1)",
  "color-bg-secondary-experimental": "rgba(243, 243, 243, 1)",
  "color-bg-input-hover-experimental": "rgba(250, 250, 250, 1)",
  "color-border-input-active-experimental": "rgba(26, 26, 26, 1)",
  "color-border-critical-strong-experimental": "rgba(142, 31, 11, 1)",
  "color-bg-input-active-experimental": "rgba(247, 247, 247, 1)",
  "color-bg-transparent-experimental": "rgba(0, 0, 0, 0)",
  "color-bg-transparent-subdued-experimental": "rgba(0, 0, 0, 0.06)",
  "color-bg-transparent-hover-experimental": "rgba(0, 0, 0, 0.05)",
  "color-bg-transparent-active-experimental": "rgba(0, 0, 0, 0.06)",
  "color-bg-transparent-disabled-experimental": "rgba(0, 0, 0, 0.05)",
  "color-bg-transparent-secondary-disabled-experimental": "rgba(0, 0, 0, 0.08)",
  "color-bg-transparent-primary-disabled-experimental": "rgba(0, 0, 0, 0.17)",
  "color-bg-transparent-primary-experimental": "rgba(0, 0, 0, 0.62)",
  "color-bg-success-strong-hover-experimental": "rgba(19, 111, 69, 1)",
  "color-bg-success-strong-active-experimental": "rgba(12, 81, 50, 1)",
  "color-bg-warning-subdued-experimental": "rgba(255, 241, 227, 1)",
  "color-bg-warning-strong-experimental": "rgba(255, 184, 0, 1)",
  "color-text-warning-experimental": "rgba(65, 45, 0, 1)",
  "color-text-critical-hover-experimental": "rgba(95, 21, 7, 1)",
  "color-icon-info-strong-experimental": "rgba(0, 82, 124, 1)",
  "color-icon-warning-strong-experimental": "rgba(124, 88, 0, 1)",
  "color-icon-success-strong-experimental": "rgba(12, 81, 50, 1)",
  "color-icon-critical-strong-experimental": "rgba(142, 31, 11, 1)",
  "color-icon-critical-strong-hover-experimental": "rgba(95, 21, 7, 1)",
  "color-icon-critical-strong-active-experimental": "rgba(47, 10, 4, 1)",
  "color-avatar-background-experimental": "rgba(181, 181, 181, 1)",
  "color-avatar-color-experimental": "rgba(255, 255, 255, 1)",
  "color-avatar-style-one-background-experimental": "rgba(251, 207, 251, 1)",
  "color-avatar-style-one-color-experimental": "rgba(121, 26, 121, 1)",
  "color-avatar-style-two-background-experimental": "rgba(56, 250, 163, 1)",
  "color-avatar-style-two-color-experimental": "rgba(12, 81, 50, 1)",
  "color-avatar-style-three-background-experimental": "rgba(146, 237, 222, 1)",
  "color-avatar-style-three-color-experimental": "rgba(9, 83, 70, 1)",
  "color-avatar-style-four-background-experimental": "rgba(192, 225, 255, 1)",
  "color-avatar-style-four-color-experimental": "rgba(0, 82, 124, 1)",
  "color-avatar-style-five-background-experimental": "rgba(254, 209, 221, 1)",
  "color-avatar-style-five-color-experimental": "rgba(141, 4, 72, 1)"
};
var font = {
  "font-family-sans": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "font-family-mono": "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
  "font-size-275": "0.6875rem",
  "font-size-325": "0.8125rem",
  "font-size-350": "0.875rem",
  "font-size-750": "1.75rem",
  "font-size-900": "2rem",
  "font-size-1000": "2.5rem",
  "font-size-70-experimental": "0.6875rem",
  "font-size-75": "0.75rem",
  "font-size-80-experimental": "0.8125rem",
  "font-size-100": "0.875rem",
  "font-size-200": "1rem",
  "font-size-300": "1.25rem",
  "font-size-400": "1.5rem",
  "font-size-500": "1.75rem",
  "font-size-600": "2rem",
  "font-size-700": "2.5rem",
  "font-weight-regular": "400",
  "font-weight-medium": "500",
  "font-weight-semibold": "600",
  "font-weight-bold": "700",
  "font-letter-spacing-densest": "-0.03375rem",
  "font-letter-spacing-denser": "-0.01875rem",
  "font-letter-spacing-dense": "-0.0125rem",
  "font-letter-spacing-normal": "0rem",
  "font-line-height-300": "0.75rem",
  "font-line-height-400": "1rem",
  "font-line-height-500": "1.25rem",
  "font-line-height-600": "1.5rem",
  "font-line-height-700": "1.75rem",
  "font-line-height-800": "2rem",
  "font-line-height-1000": "2.5rem",
  "font-line-height-1200": "3rem",
  "font-line-height-075-experimental": "0.75rem",
  "font-line-height-1": "1rem",
  "font-line-height-2": "1.25rem",
  "font-line-height-3": "1.5rem",
  "font-line-height-4": "1.75rem",
  "font-line-height-5": "2rem",
  "font-line-height-6": "2.5rem",
  "font-line-height-7": "3rem"
};
var height = {
  "height-0": "0rem",
  "height-025": "0.0625rem",
  "height-050": "0.125rem",
  "height-100": "0.25rem",
  "height-150": "0.375rem",
  "height-200": "0.5rem",
  "height-300": "0.75rem",
  "height-400": "1rem",
  "height-500": "1.25rem",
  "height-600": "1.5rem",
  "height-700": "1.75rem",
  "height-800": "2rem",
  "height-900": "2.25rem",
  "height-1000": "2.5rem",
  "height-1200": "3rem",
  "height-1600": "4rem",
  "height-2000": "5rem",
  "height-2400": "6rem",
  "height-2800": "7rem",
  "height-3200": "8rem"
};
var motion = {
  "motion-duration-0": "0ms",
  "motion-duration-50": "50ms",
  "motion-duration-100": "100ms",
  "motion-duration-150": "150ms",
  "motion-duration-200": "200ms",
  "motion-duration-250": "250ms",
  "motion-duration-300": "300ms",
  "motion-duration-350": "350ms",
  "motion-duration-400": "400ms",
  "motion-duration-450": "450ms",
  "motion-duration-500": "500ms",
  "motion-duration-5000": "5000ms",
  "motion-ease": "cubic-bezier(0.25, 0.1, 0.25, 1)",
  "motion-ease-in": "cubic-bezier(0.42, 0, 1, 1)",
  "motion-ease-out": "cubic-bezier(0, 0, 0.58, 1)",
  "motion-ease-in-out": "cubic-bezier(0.42, 0, 0.58, 1)",
  "motion-linear": "cubic-bezier(0, 0, 1, 1)",
  "motion-keyframes-bounce": "{ from, 65%, 85% { transform: scale(1) } 75% { transform: scale(0.85) } 82.5% { transform: scale(1.05) } }",
  "motion-keyframes-fade-in": "{ to { opacity: 1 } }",
  "motion-keyframes-pulse": "{ from, 75% { transform: scale(0.85); opacity: 1; } to { transform: scale(2.5); opacity: 0; } }",
  "motion-keyframes-spin": "{ to { transform: rotate(1turn) } }",
  "motion-keyframes-appear-above": "{ from { transform: translateY(var(--p-space-100)); opacity: 0; } to { transform: none; opacity: 1; } }",
  "motion-keyframes-appear-below": "{ from { transform: translateY(calc(var(--p-space-100) * -1)); opacity: 0; } to { transform: none; opacity: 1; } }"
};
var shadow = {
  "shadow-0": "none",
  "shadow-100": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
  "shadow-200": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
  "shadow-300": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
  "shadow-400": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
  "shadow-500": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
  "shadow-600": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
  "shadow-bevel-100": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
  "shadow-inset-100": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
  "shadow-inset-200": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
  "shadow-button": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
  "shadow-button-hover": "inset 0rem -0.0625rem 0rem #CCCCCC, inset 0.0625rem 0rem 0rem #EBEBEB, inset -0.0625rem 0rem 0rem #EBEBEB, inset 0rem 0.0625rem 0rem #EBEBEB",
  "shadow-button-inset": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
  "shadow-button-primary": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
  "shadow-button-primary-hover": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
  "shadow-button-primary-inset": "0rem 0.1875rem 0rem 0rem #000 inset",
  "shadow-button-primary-critical": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
  "shadow-button-primary-critical-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
  "shadow-button-primary-critical-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
  "shadow-button-primary-success": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
  "shadow-button-primary-success-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
  "shadow-button-primary-success-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
  "shadow-border-inset": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset",
  "shadow-inset-lg": "inset 0rem 0rem 0.4375rem 0.125rem rgba(31, 33, 36, 0.18)",
  "shadow-inset-md": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
  "shadow-inset-sm": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
  "shadow-none": "none",
  "shadow-xs": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
  "shadow-sm": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
  "shadow-md": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
  "shadow-lg": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
  "shadow-xl": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
  "shadow-2xl": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
  "shadow-bevel-experimental": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
  "shadow-card-sm-experimental": "0rem 0.125rem 0rem rgba(0, 0, 0, 0.07), 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.07), 0.0625rem 0rem 0rem #E3E3E3, -0.0625rem 0rem 0rem #E3E3E3, 0rem -0.0625rem 0rem #E3E3E3",
  "shadow-card-md-experimental": "0rem 0.1875rem 0.0625rem -0.0625rem rgba(0, 0, 0, 0.07),  0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.16),  0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  -0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  0rem 0.0625rem 0rem 0rem rgba(227, 227, 227, 1)",
  "shadow-card-lg-experimental": "0rem 0.25rem 0.375rem -0.125rem rgba(0, 0, 0, 0.2), inset 0rem -0.0625rem 0rem #D4D4D4, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
  "shadow-button-experimental": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
  "shadow-button-hover-experimental": "inset 0rem -0.0625rem 0rem #cccccc, inset 0.0625rem 0rem 0rem #ebebeb, inset -0.0625rem 0rem 0rem #ebebeb, inset 0rem 0.0625rem 0rem #ebebeb",
  "shadow-button-disabled-experimental": "inset 0rem 0rem 0rem 0.0625rem rgba(227, 227, 227, 1)",
  "shadow-button-primary-strong-inset-experimental": "0rem 0.1875rem 0rem 0rem #000 inset",
  "shadow-button-primary-strong-hover-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
  "shadow-button-primary-strong-experimental": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
  "shadow-button-primary-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
  "shadow-button-primary-hover-experimental": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
  "shadow-button-inset-experimental": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
  "shadow-border-inset-experimental": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset"
};
var space = {
  "space-0": "0rem",
  "space-025": "0.0625rem",
  "space-050": "0.125rem",
  "space-100": "0.25rem",
  "space-150": "0.375rem",
  "space-200": "0.5rem",
  "space-300": "0.75rem",
  "space-400": "1rem",
  "space-500": "1.25rem",
  "space-600": "1.5rem",
  "space-800": "2rem",
  "space-1000": "2.5rem",
  "space-1200": "3rem",
  "space-1600": "4rem",
  "space-2000": "5rem",
  "space-2400": "6rem",
  "space-2800": "7rem",
  "space-3200": "8rem",
  "space-button-group-gap": "0.5rem",
  "space-card-gap": "1rem",
  "space-card-padding": "1rem",
  "space-table-cell-padding": "0.375rem",
  "space-05": "0.125rem",
  "space-1": "0.25rem",
  "space-1_5-experimental": "0.375rem",
  "space-2": "0.5rem",
  "space-3": "0.75rem",
  "space-4": "1rem",
  "space-5": "1.25rem",
  "space-6": "1.5rem",
  "space-8": "2rem",
  "space-10": "2.5rem",
  "space-12": "3rem",
  "space-16": "4rem",
  "space-20": "5rem",
  "space-24": "6rem",
  "space-28": "7rem",
  "space-32": "8rem"
};
var text = {
  "text-heading-3xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-heading-3xl-font-size": "2rem",
  "text-heading-3xl-font-weight": "700",
  "text-heading-3xl-font-letter-spacing": "-0.03375rem",
  "text-heading-3xl-font-line-height": "3rem",
  "text-heading-2xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-heading-2xl-font-size": "1.75rem",
  "text-heading-2xl-font-weight": "700",
  "text-heading-2xl-font-letter-spacing": "-0.01875rem",
  "text-heading-2xl-font-line-height": "2.5rem",
  "text-heading-xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-heading-xl-font-size": "2rem",
  "text-heading-xl-font-weight": "700",
  "text-heading-xl-font-letter-spacing": "-0.0125rem",
  "text-heading-xl-font-line-height": "2rem",
  "text-heading-lg-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-heading-lg-font-size": "1.75rem",
  "text-heading-lg-font-weight": "600",
  "text-heading-lg-font-letter-spacing": "-0.0125rem",
  "text-heading-lg-font-line-height": "1.5rem",
  "text-heading-md-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-heading-md-font-size": "0.875rem",
  "text-heading-md-font-weight": "600",
  "text-heading-md-font-letter-spacing": "0rem",
  "text-heading-md-font-line-height": "1.25rem",
  "text-heading-sm-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-heading-sm-font-size": "0.8125rem",
  "text-heading-sm-font-weight": "600",
  "text-heading-sm-font-letter-spacing": "0rem",
  "text-heading-sm-font-line-height": "1.25rem",
  "text-body-lg-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-body-lg-font-size": "0.875rem",
  "text-body-lg-font-weight": "400",
  "text-body-lg-font-letter-spacing": "0rem",
  "text-body-lg-font-line-height": "1.25rem",
  "text-body-md-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-body-md-font-size": "0.8125rem",
  "text-body-md-font-weight": "400",
  "text-body-md-font-letter-spacing": "0rem",
  "text-body-md-font-line-height": "1.25rem",
  "text-body-sm-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-body-sm-font-size": "1.25rem",
  "text-body-sm-font-weight": "400",
  "text-body-sm-font-letter-spacing": "0rem",
  "text-body-sm-font-line-height": "1rem",
  "text-body-xs-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
  "text-body-xs-font-size": "0.6875rem",
  "text-body-xs-font-weight": "400",
  "text-body-xs-font-letter-spacing": "0rem",
  "text-body-xs-font-line-height": "0.75rem"
};
var width = {
  "width-0": "0rem",
  "width-025": "0.0625rem",
  "width-050": "0.125rem",
  "width-100": "0.25rem",
  "width-150": "0.375rem",
  "width-200": "0.5rem",
  "width-300": "0.75rem",
  "width-400": "1rem",
  "width-500": "1.25rem",
  "width-600": "1.5rem",
  "width-700": "1.75rem",
  "width-800": "2rem",
  "width-900": "2.25rem",
  "width-1000": "2.5rem",
  "width-1200": "3rem",
  "width-1600": "4rem",
  "width-2000": "5rem",
  "width-2400": "6rem",
  "width-2800": "7rem",
  "width-3200": "8rem"
};
var zIndex = {
  "z-index-1": "100",
  "z-index-2": "400",
  "z-index-3": "510",
  "z-index-4": "512",
  "z-index-5": "513",
  "z-index-6": "514",
  "z-index-7": "515",
  "z-index-8": "516",
  "z-index-9": "517",
  "z-index-10": "518",
  "z-index-11": "519",
  "z-index-12": "520"
};
var tokens = {
  "border": {
    "border-radius-0": "0rem",
    "border-radius-050": "0.125rem",
    "border-radius-100": "0.25rem",
    "border-radius-150": "0.375rem",
    "border-radius-200": "0.5rem",
    "border-radius-300": "0.75rem",
    "border-radius-400": "1rem",
    "border-radius-500": "1.25rem",
    "border-radius-750": "1.875rem",
    "border-radius-full": "624.9375rem",
    "border-radius-0-experimental": "0rem",
    "border-radius-05": "0.125rem",
    "border-radius-1": "0.25rem",
    "border-radius-2": "0.5rem",
    "border-radius-3": "0.75rem",
    "border-radius-4": "1rem",
    "border-radius-5": "1.25rem",
    "border-radius-6": "1.875rem",
    "border-radius-1_5-experimental": "0.375rem",
    "border-width-0165": "0.04125rem",
    "border-width-025": "0.0625rem",
    "border-width-050": "0.125rem",
    "border-width-100": "0.25rem",
    "border-width-1": "0.0625rem",
    "border-width-2": "0.125rem",
    "border-width-3": "0.1875rem",
    "border-width-4": "0.25rem",
    "border-width-5": "0.3125rem",
    "border-width-1-experimental": "0.04125rem",
    "border-width-2-experimental": "0.0625rem"
  },
  "breakpoints": {
    "breakpoints-xs": "0rem",
    "breakpoints-sm": "30.625rem",
    "breakpoints-md": "48rem",
    "breakpoints-lg": "65rem",
    "breakpoints-xl": "90rem"
  },
  "color": {
    "color-border-inverse-active": "rgba(227, 227, 227, 1)",
    "color-border-inverse-hover": "rgba(204, 204, 204, 1)",
    "color-border-warning": "rgba(255, 200, 121, 1)",
    "color-checkbox-icon-disabled": "rgba(255, 255, 255, 1)",
    "color-checkbox-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
    "color-bg-fill-active": "rgba(247, 247, 247, 1)",
    "color-bg-fill-brand-selected": "rgba(48, 48, 48, 1)",
    "color-bg-fill-caution-active": "rgba(225, 203, 0, 1)",
    "color-bg-fill-caution-hover": "rgba(234, 211, 0, 1)",
    "color-bg-fill-critical-selected": "rgba(142, 31, 11, 1)",
    "color-bg-fill-emphasis-active": "rgba(0, 46, 106, 1)",
    "color-bg-fill-emphasis-hover": "rgba(0, 66, 153, 1)",
    "color-bg-fill-emphasis": "rgba(0, 91, 211, 1)",
    "color-bg-fill-hover": "rgba(250, 250, 250, 1)",
    "color-bg-fill-info-active": "rgba(0, 148, 213, 1)",
    "color-bg-fill-info-hover": "rgba(81, 192, 255, 1)",
    "color-bg-fill-secondary-active": "rgba(227, 227, 227, 1)",
    "color-bg-fill-secondary-hover": "rgba(235, 235, 235, 1)",
    "color-bg-fill-selected": "rgba(204, 204, 204, 1)",
    "color-bg-fill-transparent-secondary-active": "rgba(0, 0, 0, 0.11)",
    "color-bg-fill-transparent-secondary-hover": "rgba(0, 0, 0, 0.08)",
    "color-bg-fill-transparent-selected": "rgba(0, 0, 0, 0.06)",
    "color-bg-fill-transparent": "rgba(0, 0, 0, 0)",
    "color-bg-fill-warning-active": "rgba(178, 132, 0, 1)",
    "color-bg-fill-warning-hover": "rgba(229, 165, 0, 1)",
    "color-bg-fill": "rgba(255, 255, 255, 1)",
    "color-icon-secondary-active": "rgba(74, 74, 74, 1)",
    "color-icon-secondary-hover": "rgba(97, 97, 97, 1)",
    "color-text-link-active": "rgba(0, 46, 106, 1)",
    "color-text-link-hover": "rgba(0, 66, 153, 1)",
    "color-text-link": "rgba(0, 91, 211, 1)",
    "color-nav-bg": "rgba(235, 235, 235, 1)",
    "color-nav-bg-surface-active": "rgba(250, 250, 250, 1)",
    "color-nav-bg-surface-hover": "rgba(241, 241, 241, 1)",
    "color-nav-bg-surface-selected": "rgba(250, 250, 250, 1)",
    "color-nav-bg-surface": "rgba(0, 0, 0, 0.02)",
    "color-radio-button-icon-disabled": "rgba(255, 255, 255, 1)",
    "color-radio-button-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
    "color-bg-surface-emphasis-active": "rgba(226, 231, 255, 1)",
    "color-bg-surface-emphasis-hover": "rgba(234, 237, 255, 1)",
    "color-bg-surface-emphasis": "rgba(240, 242, 255, 1)",
    "color-bg-surface-inverse": "rgba(48, 48, 48, 1)",
    "color-bg-surface-magic-active": "rgba(228, 222, 255, 1)",
    "color-bg-surface-secondary-selected": "rgba(235, 235, 235, 1)",
    "color-bg-surface-tertiary-active": "rgba(227, 227, 227, 1)",
    "color-bg-surface-tertiary-hover": "rgba(235, 235, 235, 1)",
    "color-bg-surface-warning-active": "rgba(255, 228, 198, 1)",
    "color-bg-surface-warning-hover": "rgba(255, 235, 213, 1)",
    "color-text-brand-on-bg-fill-active": "rgba(204, 204, 204, 1)",
    "color-text-brand-on-bg-fill-disabled": "rgba(255, 255, 255, 1)",
    "color-text-brand-on-bg-fill-hover": "rgba(227, 227, 227, 1)",
    "color-text-brand-on-bg-fill": "rgba(255, 255, 255, 1)",
    "color-text-caution-active": "rgba(31, 28, 0, 1)",
    "color-text-caution-hover": "rgba(51, 46, 0, 1)",
    "color-text-caution-on-bg-fill": "rgba(51, 46, 0, 1)",
    "color-text-critical-on-bg-fill": "rgba(255, 251, 251, 1)",
    "color-text-emphasis-on-bg-fill-active": "rgba(213, 220, 255, 1)",
    "color-text-emphasis-on-bg-fill-hover": "rgba(226, 231, 255, 1)",
    "color-text-emphasis-on-bg-fill": "rgba(252, 253, 255, 1)",
    "color-text-info-active": "rgba(0, 33, 51, 1)",
    "color-text-info-hover": "rgba(0, 58, 90, 1)",
    "color-text-magic-on-bg-fill": "rgba(253, 253, 255, 1)",
    "color-text-success-active": "rgba(9, 42, 27, 1)",
    "color-text-success-hover": "rgba(8, 61, 37, 1)",
    "color-text-success-on-bg-fill": "rgba(248, 255, 251, 1)",
    "color-text-warning-active": "rgba(37, 26, 0, 1)",
    "color-text-warning-hover": "rgba(65, 45, 0, 1)",
    "color-text-warning-on-bg-fill": "rgba(37, 26, 0, 1)",
    "color-video-thumbnail-play-button-bg-fill-hover": "rgba(0, 0, 0, 0.81)",
    "color-video-thumbnail-play-button-bg-fill": "rgba(0, 0, 0, 0.71)",
    "color-video-thumbnail-play-button-text-on-bg-fill": "rgba(255, 255, 255, 1)",
    "color-bg-surface": "rgba(255, 255, 255, 1)",
    "color-bg-surface-hover": "rgba(247, 247, 247, 1)",
    "color-bg-surface-active": "rgba(243, 243, 243, 1)",
    "color-bg-surface-disabled": "rgba(0, 0, 0, 0.05)",
    "color-bg-surface-secondary": "rgba(249, 250, 251, 1)",
    "color-bg-surface-secondary-hover": "rgba(241, 242, 244, 1)",
    "color-bg-surface-secondary-active": "rgba(235, 236, 239, 1)",
    "color-bg-surface-tertiary": "rgba(243, 243, 243, 1)",
    "color-bg-fill-tertiary": "rgba(221, 224, 228, 1)",
    "color-bg-fill-tertiary-hover": "rgba(202, 206, 211, 1)",
    "color-bg-fill-tertiary-active": "rgba(202, 206, 211, 1)",
    "color-input-bg-surface": "rgba(255, 255, 255, 1)",
    "color-input-bg-surface-hover": "rgba(250, 250, 250, 1)",
    "color-input-bg-surface-active": "rgba(247, 247, 247, 1)",
    "color-bg-fill-brand": "rgba(48, 48, 48, 1)",
    "color-bg-fill-brand-hover": "rgba(26, 26, 26, 1)",
    "color-bg-fill-brand-active": "rgba(26, 26, 26, 1)",
    "color-bg-surface-brand": "rgba(227, 227, 227, 1)",
    "color-bg-surface-brand-hover": "rgba(235, 235, 235, 1)",
    "color-bg-surface-brand-active": "rgba(241, 241, 241, 1)",
    "color-bg-surface-brand-selected": "rgba(241, 241, 241, 1)",
    "color-bg-surface-selected": "rgba(235, 236, 239, 1)",
    "color-bg-fill-success": "rgba(22, 166, 121, 1)",
    "color-bg-fill-success-hover": "rgba(19, 111, 69, 1)",
    "color-bg-fill-success-active": "rgba(12, 81, 50, 1)",
    "color-bg-fill-success-secondary": "rgba(161, 237, 208, 1)",
    "color-bg-surface-success": "rgba(224, 248, 238, 1)",
    "color-bg-surface-success-hover": "rgba(240, 253, 248, 1)",
    "color-bg-surface-success-active": "rgba(192, 242, 221, 1)",
    "color-bg-fill-critical": "rgba(197, 40, 12, 1)",
    "color-bg-fill-critical-hover": "rgba(159, 32, 10, 1)",
    "color-bg-fill-critical-active": "rgba(115, 24, 7, 1)",
    "color-bg-fill-critical-secondary": "rgba(251, 197, 188, 1)",
    "color-bg-surface-critical": "rgba(253, 226, 221, 1)",
    "color-bg-surface-critical-hover": "rgba(254, 243, 241, 1)",
    "color-bg-surface-critical-active": "rgba(251, 197, 188, 1)",
    "color-bg-fill-caution": "rgba(216, 155, 13, 1)",
    "color-bg-fill-caution-secondary": "rgba(248, 217, 144, 1)",
    "color-bg-surface-caution": "rgba(252, 240, 212, 1)",
    "color-bg-surface-caution-hover": "rgba(254, 248, 236, 1)",
    "color-bg-surface-caution-active": "rgba(250, 229, 178, 1)",
    "color-bg-fill-info": "rgba(42, 172, 187, 1)",
    "color-bg-fill-info-secondary": "rgba(184, 233, 239, 1)",
    "color-bg-surface-info": "rgba(222, 245, 247, 1)",
    "color-bg-surface-info-hover": "rgba(238, 250, 251, 1)",
    "color-bg-surface-info-active": "rgba(184, 233, 239, 1)",
    "color-bg-fill-warning": "rgba(255, 184, 0, 1)",
    "color-bg-fill-warning-secondary": "rgba(250, 201, 168, 1)",
    "color-bg-surface-warning": "rgba(255, 241, 227, 1)",
    "color-bg-fill-magic": "rgba(121, 69, 227, 1)",
    "color-bg-fill-magic-secondary": "rgba(236, 227, 253, 1)",
    "color-bg-fill-magic-secondary-hover": "rgba(226, 214, 250, 1)",
    "color-bg-fill-magic-secondary-active": "rgba(203, 180, 248, 1)",
    "color-bg-surface-magic": "rgba(243, 241, 255, 1)",
    "color-bg-surface-magic-hover": "rgba(236, 227, 253, 1)",
    "color-bg-fill-secondary": "rgba(241, 242, 244, 1)",
    "color-bg-fill-inverse": "rgba(97, 106, 117, 1)",
    "color-bg-fill-inverse-hover": "rgba(97, 106, 117, 1)",
    "color-bg-fill-inverse-active": "rgba(135, 144, 155, 1)",
    "color-bg-surface-transparent": "rgba(0, 0, 0, 0)",
    "color-bg-fill-transparent-hover": "rgba(0, 0, 0, 0.05)",
    "color-bg-fill-transparent-active": "rgba(0, 0, 0, 0.06)",
    "color-bg-fill-disabled": "rgba(0, 0, 0, 0.05)",
    "color-bg-fill-transparent-secondary": "rgba(0, 0, 0, 0.06)",
    "color-bg-fill-brand-disabled": "rgba(0, 0, 0, 0.17)",
    "color-backdrop-bg": "rgba(0, 0, 0, 0.71)",
    "color-avatar-bg-fill": "rgba(181, 181, 181, 1)",
    "color-avatar-one-bg-fill": "rgba(251, 207, 251, 1)",
    "color-avatar-two-bg-fill": "rgba(56, 250, 163, 1)",
    "color-avatar-three-bg-fill": "rgba(146, 237, 222, 1)",
    "color-avatar-four-bg-fill": "rgba(192, 225, 255, 1)",
    "color-avatar-five-bg-fill": "rgba(254, 209, 221, 1)",
    "color-text-secondary": "rgba(97, 106, 117, 1)",
    "color-text-emphasis": "rgba(36, 99, 188, 1)",
    "color-text-emphasis-hover": "rgba(20, 73, 149, 1)",
    "color-text-emphasis-active": "rgba(14, 53, 108, 1)",
    "color-text-brand": "rgba(0, 122, 92, 1)",
    "color-text-brand-hover": "rgba(18, 84, 67, 1)",
    "color-text-critical-hover": "rgba(95, 21, 7, 1)",
    "color-text-info-on-bg-fill": "rgba(16, 65, 71, 1)",
    "color-text-warning": "rgba(94, 66, 0, 1)",
    "color-text-inverse-secondary": "rgba(171, 177, 186, 1)",
    "color-text-link-inverse": "rgba(197, 208, 255, 1)",
    "color-avatar-text-on-bg-fill": "rgba(255, 255, 255, 1)",
    "color-avatar-one-text-on-bg-fill": "rgba(121, 26, 121, 1)",
    "color-avatar-two-text-on-bg-fill": "rgba(12, 81, 50, 1)",
    "color-avatar-three-text-on-bg-fill": "rgba(9, 83, 70, 1)",
    "color-avatar-four-text-on-bg-fill": "rgba(0, 82, 124, 1)",
    "color-avatar-five-text-on-bg-fill": "rgba(141, 4, 72, 1)",
    "color-icon-secondary": "rgba(135, 144, 155, 1)",
    "color-icon-emphasis": "rgba(36, 99, 188, 1)",
    "color-icon-emphasis-hover": "rgba(20, 73, 149, 1)",
    "color-icon-emphasis-active": "rgba(14, 53, 108, 1)",
    "color-icon-brand": "rgba(0, 122, 92, 1)",
    "color-border-secondary": "rgba(221, 224, 228, 1)",
    "color-border-tertiary": "rgba(204, 204, 204, 1)",
    "color-input-border": "rgba(171, 177, 186, 1)",
    "color-input-border-hover": "rgba(97, 106, 117, 1)",
    "color-input-border-active": "rgba(26, 26, 26, 1)",
    "color-border-emphasis": "rgba(0, 91, 211, 1)",
    "color-border-emphasis-hover": "rgba(20, 73, 149, 1)",
    "color-border-emphasis-active": "rgba(14, 53, 108, 1)",
    "color-border-focus": "rgba(62, 125, 213, 1)",
    "color-border-brand": "rgba(0, 122, 92, 1)",
    "color-border-critical-secondary": "rgba(142, 31, 11, 1)",
    "color-border-magic-secondary": "rgba(121, 69, 227, 1)",
    "color-bg-inverse": "rgba(31, 33, 36, 1)",
    "color-bg-inset-strong": "rgba(97, 106, 117, 1)",
    "color-bg-inverse-hover": "rgba(97, 106, 117, 1)",
    "color-bg-inverse-active": "rgba(135, 144, 155, 1)",
    "color-bg-strong-hover": "rgba(202, 206, 211, 1)",
    "color-bg-strong-active": "rgba(202, 206, 211, 1)",
    "color-bg-strong": "rgba(221, 224, 228, 1)",
    "color-bg-subdued-active": "rgba(235, 236, 239, 1)",
    "color-bg-disabled": "rgba(235, 236, 239, 1)",
    "color-bg-interactive-disabled": "rgba(235, 236, 239, 1)",
    "color-bg-app": "rgba(241, 242, 244, 1)",
    "color-bg-app-active": "rgba(221, 224, 228, 1)",
    "color-bg-app-hover": "rgba(235, 236, 239, 1)",
    "color-bg-app-selected": "rgba(235, 236, 239, 1)",
    "color-bg-active": "rgba(235, 236, 239, 1)",
    "color-bg-subdued-hover": "rgba(241, 242, 244, 1)",
    "color-bg-inset": "rgba(241, 242, 244, 1)",
    "color-bg-hover": "rgba(241, 242, 244, 1)",
    "color-bg-subdued": "rgba(249, 250, 251, 1)",
    "color-bg-input": "rgba(255, 255, 255, 1)",
    "color-bg": "rgba(255, 255, 255, 1)",
    "color-bg-primary-active": "rgba(12, 59, 47, 1)",
    "color-bg-primary-hover": "rgba(18, 84, 67, 1)",
    "color-bg-primary": "rgba(0, 122, 92, 1)",
    "color-bg-success-strong": "rgba(22, 166, 121, 1)",
    "color-bg-success": "rgba(161, 237, 208, 1)",
    "color-bg-primary-subdued-active": "rgba(192, 242, 221, 1)",
    "color-bg-success-subdued-active": "rgba(192, 242, 221, 1)",
    "color-bg-success-subdued": "rgba(224, 248, 238, 1)",
    "color-bg-primary-subdued-hover": "rgba(224, 248, 238, 1)",
    "color-bg-success-subdued-hover": "rgba(240, 253, 248, 1)",
    "color-bg-primary-subdued": "rgba(240, 253, 248, 1)",
    "color-bg-primary-subdued-selected": "rgba(240, 253, 248, 1)",
    "color-bg-critical-strong-active": "rgba(115, 24, 7, 1)",
    "color-bg-critical-strong-hover": "rgba(159, 32, 10, 1)",
    "color-bg-critical-strong": "rgba(197, 40, 12, 1)",
    "color-bg-critical-subdued-active": "rgba(251, 197, 188, 1)",
    "color-bg-critical": "rgba(251, 197, 188, 1)",
    "color-bg-critical-subdued": "rgba(253, 226, 221, 1)",
    "color-bg-critical-subdued-hover": "rgba(254, 243, 241, 1)",
    "color-bg-caution-strong": "rgba(216, 155, 13, 1)",
    "color-bg-caution": "rgba(248, 217, 144, 1)",
    "color-bg-caution-subdued-active": "rgba(250, 229, 178, 1)",
    "color-bg-caution-subdued": "rgba(252, 240, 212, 1)",
    "color-bg-caution-subdued-hover": "rgba(254, 248, 236, 1)",
    "color-bg-info-strong": "rgba(42, 172, 187, 1)",
    "color-bg-info-subdued-active": "rgba(184, 233, 239, 1)",
    "color-bg-info": "rgba(184, 233, 239, 1)",
    "color-bg-info-subdued": "rgba(222, 245, 247, 1)",
    "color-bg-info-subdued-hover": "rgba(238, 250, 251, 1)",
    "color-bg-interactive-active": "rgba(14, 53, 108, 1)",
    "color-bg-interactive-hover": "rgba(20, 73, 149, 1)",
    "color-bg-interactive": "rgba(36, 99, 188, 1)",
    "color-bg-interactive-subdued-active": "rgba(187, 212, 247, 1)",
    "color-bg-interactive-subdued-hover": "rgba(232, 240, 253, 1)",
    "color-bg-interactive-subdued": "rgba(240, 245, 253, 1)",
    "color-bg-interactive-selected": "rgba(240, 245, 253, 1)",
    "color-bg-warning": "rgba(250, 201, 168, 1)",
    "color-bg-magic-strong": "rgba(121, 69, 227, 1)",
    "color-bg-magic-hover": "rgba(226, 214, 250, 1)",
    "color-bg-magic-active": "rgba(203, 180, 248, 1)",
    "color-bg-magic": "rgba(236, 227, 253, 1)",
    "color-bg-magic-subdued-hover": "rgba(236, 227, 253, 1)",
    "color-bg-magic-subdued-active": "rgba(226, 214, 250, 1)",
    "color-bg-magic-subdued": "rgba(242, 237, 253, 1)",
    "color-border-input-hover": "rgba(97, 106, 117, 1)",
    "color-border-inverse": "rgba(97, 106, 117, 1)",
    "color-border-strong-hover": "rgba(135, 144, 155, 1)",
    "color-border-input": "rgba(171, 177, 186, 1)",
    "color-border-hover": "rgba(171, 177, 186, 1)",
    "color-border-strong": "rgba(171, 177, 186, 1)",
    "color-border": "rgba(202, 206, 211, 1)",
    "color-border-disabled": "rgba(221, 224, 228, 1)",
    "color-border-subdued": "rgba(221, 224, 228, 1)",
    "color-border-interactive-disabled": "rgba(221, 224, 228, 1)",
    "color-border-primary": "rgba(0, 122, 92, 1)",
    "color-border-success": "rgba(22, 166, 121, 1)",
    "color-border-success-subdued": "rgba(80, 220, 169, 1)",
    "color-border-critical-active": "rgba(67, 14, 4, 1)",
    "color-border-critical-hover": "rgba(115, 24, 7, 1)",
    "color-border-critical": "rgba(197, 40, 12, 1)",
    "color-border-critical-subdued": "rgba(245, 107, 82, 1)",
    "color-border-caution": "rgba(216, 155, 13, 1)",
    "color-border-caution-subdued": "rgba(245, 196, 82, 1)",
    "color-border-info": "rgba(59, 195, 211, 1)",
    "color-border-info-subdued": "rgba(109, 211, 222, 1)",
    "color-border-interactive-active": "rgba(14, 53, 108, 1)",
    "color-border-interactive-hover": "rgba(20, 73, 149, 1)",
    "color-border-interactive": "rgba(62, 125, 213, 1)",
    "color-border-interactive-focus": "rgba(62, 125, 213, 1)",
    "color-border-interactive-subdued": "rgba(187, 212, 247, 1)",
    "color-border-magic-strong": "rgba(121, 69, 227, 1)",
    "color-border-magic": "rgba(173, 139, 241, 1)",
    "color-icon-hover": "rgba(31, 33, 36, 1)",
    "color-icon": "rgba(97, 106, 117, 1)",
    "color-icon-active": "rgba(31, 33, 36, 1)",
    "color-icon-subdued": "rgba(135, 144, 155, 1)",
    "color-icon-disabled": "rgba(171, 177, 186, 1)",
    "color-icon-interactive-disabled": "rgba(171, 177, 186, 1)",
    "color-icon-inverse": "rgba(221, 224, 228, 1)",
    "color-icon-on-color": "rgba(255, 255, 255, 1)",
    "color-icon-primary": "rgba(0, 122, 92, 1)",
    "color-icon-success": "rgba(22, 166, 121, 1)",
    "color-icon-critical": "rgba(197, 40, 12, 1)",
    "color-icon-caution": "rgba(183, 126, 11, 1)",
    "color-icon-info": "rgba(42, 172, 187, 1)",
    "color-icon-warning": "rgba(242, 117, 34, 1)",
    "color-icon-interactive-active": "rgba(14, 53, 108, 1)",
    "color-icon-interactive-hover": "rgba(20, 73, 149, 1)",
    "color-icon-interactive": "rgba(36, 99, 188, 1)",
    "color-icon-interactive-inverse": "rgba(102, 153, 225, 1)",
    "color-icon-magic": "rgba(121, 69, 227, 1)",
    "color-text": "rgba(31, 33, 36, 1)",
    "color-text-subdued": "rgba(97, 106, 117, 1)",
    "color-text-disabled": "rgba(135, 144, 155, 1)",
    "color-text-interactive-disabled": "rgba(135, 144, 155, 1)",
    "color-text-inverse-subdued": "rgba(171, 177, 186, 1)",
    "color-text-inverse": "rgba(241, 242, 244, 1)",
    "color-text-on-color": "rgba(255, 255, 255, 1)",
    "color-text-success-strong": "rgba(12, 59, 47, 1)",
    "color-text-success": "rgba(0, 122, 92, 1)",
    "color-text-primary": "rgba(0, 122, 92, 1)",
    "color-text-primary-hover": "rgba(18, 84, 67, 1)",
    "color-text-critical-strong": "rgba(67, 14, 4, 1)",
    "color-text-critical-active": "rgba(115, 24, 7, 1)",
    "color-text-critical": "rgba(197, 40, 12, 1)",
    "color-text-caution-strong": "rgba(77, 46, 5, 1)",
    "color-text-caution": "rgba(135, 92, 8, 1)",
    "color-text-info-strong": "rgba(16, 65, 71, 1)",
    "color-text-info": "rgba(32, 130, 141, 1)",
    "color-text-warning-strong": "rgba(77, 36, 5, 1)",
    "color-text-interactive-active": "rgba(14, 53, 108, 1)",
    "color-text-interactive-hover": "rgba(20, 73, 149, 1)",
    "color-text-interactive": "rgba(36, 99, 188, 1)",
    "color-text-interactive-inverse": "rgba(102, 153, 225, 1)",
    "color-text-magic-strong": "rgba(49, 13, 120, 1)",
    "color-text-magic": "rgba(90, 36, 205, 1)",
    "color-bg-backdrop-experimental": "rgba(0, 0, 0, 0.71)",
    "color-bg-primary-disabled-experimental": "rgba(212, 212, 212, 1)",
    "color-bg-secondary-experimental": "rgba(243, 243, 243, 1)",
    "color-bg-input-hover-experimental": "rgba(250, 250, 250, 1)",
    "color-border-input-active-experimental": "rgba(26, 26, 26, 1)",
    "color-border-critical-strong-experimental": "rgba(142, 31, 11, 1)",
    "color-bg-input-active-experimental": "rgba(247, 247, 247, 1)",
    "color-bg-transparent-experimental": "rgba(0, 0, 0, 0)",
    "color-bg-transparent-subdued-experimental": "rgba(0, 0, 0, 0.06)",
    "color-bg-transparent-hover-experimental": "rgba(0, 0, 0, 0.05)",
    "color-bg-transparent-active-experimental": "rgba(0, 0, 0, 0.06)",
    "color-bg-transparent-disabled-experimental": "rgba(0, 0, 0, 0.05)",
    "color-bg-transparent-secondary-disabled-experimental": "rgba(0, 0, 0, 0.08)",
    "color-bg-transparent-primary-disabled-experimental": "rgba(0, 0, 0, 0.17)",
    "color-bg-transparent-primary-experimental": "rgba(0, 0, 0, 0.62)",
    "color-bg-success-strong-hover-experimental": "rgba(19, 111, 69, 1)",
    "color-bg-success-strong-active-experimental": "rgba(12, 81, 50, 1)",
    "color-bg-warning-subdued-experimental": "rgba(255, 241, 227, 1)",
    "color-bg-warning-strong-experimental": "rgba(255, 184, 0, 1)",
    "color-text-warning-experimental": "rgba(65, 45, 0, 1)",
    "color-text-critical-hover-experimental": "rgba(95, 21, 7, 1)",
    "color-icon-info-strong-experimental": "rgba(0, 82, 124, 1)",
    "color-icon-warning-strong-experimental": "rgba(124, 88, 0, 1)",
    "color-icon-success-strong-experimental": "rgba(12, 81, 50, 1)",
    "color-icon-critical-strong-experimental": "rgba(142, 31, 11, 1)",
    "color-icon-critical-strong-hover-experimental": "rgba(95, 21, 7, 1)",
    "color-icon-critical-strong-active-experimental": "rgba(47, 10, 4, 1)",
    "color-avatar-background-experimental": "rgba(181, 181, 181, 1)",
    "color-avatar-color-experimental": "rgba(255, 255, 255, 1)",
    "color-avatar-style-one-background-experimental": "rgba(251, 207, 251, 1)",
    "color-avatar-style-one-color-experimental": "rgba(121, 26, 121, 1)",
    "color-avatar-style-two-background-experimental": "rgba(56, 250, 163, 1)",
    "color-avatar-style-two-color-experimental": "rgba(12, 81, 50, 1)",
    "color-avatar-style-three-background-experimental": "rgba(146, 237, 222, 1)",
    "color-avatar-style-three-color-experimental": "rgba(9, 83, 70, 1)",
    "color-avatar-style-four-background-experimental": "rgba(192, 225, 255, 1)",
    "color-avatar-style-four-color-experimental": "rgba(0, 82, 124, 1)",
    "color-avatar-style-five-background-experimental": "rgba(254, 209, 221, 1)",
    "color-avatar-style-five-color-experimental": "rgba(141, 4, 72, 1)"
  },
  "font": {
    "font-family-sans": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "font-family-mono": "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
    "font-size-275": "0.6875rem",
    "font-size-325": "0.8125rem",
    "font-size-350": "0.875rem",
    "font-size-750": "1.75rem",
    "font-size-900": "2rem",
    "font-size-1000": "2.5rem",
    "font-size-70-experimental": "0.6875rem",
    "font-size-75": "0.75rem",
    "font-size-80-experimental": "0.8125rem",
    "font-size-100": "0.875rem",
    "font-size-200": "1rem",
    "font-size-300": "1.25rem",
    "font-size-400": "1.5rem",
    "font-size-500": "1.75rem",
    "font-size-600": "2rem",
    "font-size-700": "2.5rem",
    "font-weight-regular": "400",
    "font-weight-medium": "500",
    "font-weight-semibold": "600",
    "font-weight-bold": "700",
    "font-letter-spacing-densest": "-0.03375rem",
    "font-letter-spacing-denser": "-0.01875rem",
    "font-letter-spacing-dense": "-0.0125rem",
    "font-letter-spacing-normal": "0rem",
    "font-line-height-300": "0.75rem",
    "font-line-height-400": "1rem",
    "font-line-height-500": "1.25rem",
    "font-line-height-600": "1.5rem",
    "font-line-height-700": "1.75rem",
    "font-line-height-800": "2rem",
    "font-line-height-1000": "2.5rem",
    "font-line-height-1200": "3rem",
    "font-line-height-075-experimental": "0.75rem",
    "font-line-height-1": "1rem",
    "font-line-height-2": "1.25rem",
    "font-line-height-3": "1.5rem",
    "font-line-height-4": "1.75rem",
    "font-line-height-5": "2rem",
    "font-line-height-6": "2.5rem",
    "font-line-height-7": "3rem"
  },
  "height": {
    "height-0": "0rem",
    "height-025": "0.0625rem",
    "height-050": "0.125rem",
    "height-100": "0.25rem",
    "height-150": "0.375rem",
    "height-200": "0.5rem",
    "height-300": "0.75rem",
    "height-400": "1rem",
    "height-500": "1.25rem",
    "height-600": "1.5rem",
    "height-700": "1.75rem",
    "height-800": "2rem",
    "height-900": "2.25rem",
    "height-1000": "2.5rem",
    "height-1200": "3rem",
    "height-1600": "4rem",
    "height-2000": "5rem",
    "height-2400": "6rem",
    "height-2800": "7rem",
    "height-3200": "8rem"
  },
  "motion": {
    "motion-duration-0": "0ms",
    "motion-duration-50": "50ms",
    "motion-duration-100": "100ms",
    "motion-duration-150": "150ms",
    "motion-duration-200": "200ms",
    "motion-duration-250": "250ms",
    "motion-duration-300": "300ms",
    "motion-duration-350": "350ms",
    "motion-duration-400": "400ms",
    "motion-duration-450": "450ms",
    "motion-duration-500": "500ms",
    "motion-duration-5000": "5000ms",
    "motion-ease": "cubic-bezier(0.25, 0.1, 0.25, 1)",
    "motion-ease-in": "cubic-bezier(0.42, 0, 1, 1)",
    "motion-ease-out": "cubic-bezier(0, 0, 0.58, 1)",
    "motion-ease-in-out": "cubic-bezier(0.42, 0, 0.58, 1)",
    "motion-linear": "cubic-bezier(0, 0, 1, 1)",
    "motion-keyframes-bounce": "{ from, 65%, 85% { transform: scale(1) } 75% { transform: scale(0.85) } 82.5% { transform: scale(1.05) } }",
    "motion-keyframes-fade-in": "{ to { opacity: 1 } }",
    "motion-keyframes-pulse": "{ from, 75% { transform: scale(0.85); opacity: 1; } to { transform: scale(2.5); opacity: 0; } }",
    "motion-keyframes-spin": "{ to { transform: rotate(1turn) } }",
    "motion-keyframes-appear-above": "{ from { transform: translateY(var(--p-space-100)); opacity: 0; } to { transform: none; opacity: 1; } }",
    "motion-keyframes-appear-below": "{ from { transform: translateY(calc(var(--p-space-100) * -1)); opacity: 0; } to { transform: none; opacity: 1; } }"
  },
  "shadow": {
    "shadow-0": "none",
    "shadow-100": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
    "shadow-200": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
    "shadow-300": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
    "shadow-400": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
    "shadow-500": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
    "shadow-600": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
    "shadow-bevel-100": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
    "shadow-inset-100": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
    "shadow-inset-200": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
    "shadow-button": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
    "shadow-button-hover": "inset 0rem -0.0625rem 0rem #CCCCCC, inset 0.0625rem 0rem 0rem #EBEBEB, inset -0.0625rem 0rem 0rem #EBEBEB, inset 0rem 0.0625rem 0rem #EBEBEB",
    "shadow-button-inset": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
    "shadow-button-primary": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
    "shadow-button-primary-hover": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
    "shadow-button-primary-inset": "0rem 0.1875rem 0rem 0rem #000 inset",
    "shadow-button-primary-critical": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
    "shadow-button-primary-critical-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
    "shadow-button-primary-critical-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
    "shadow-button-primary-success": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
    "shadow-button-primary-success-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
    "shadow-button-primary-success-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
    "shadow-border-inset": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset",
    "shadow-inset-lg": "inset 0rem 0rem 0.4375rem 0.125rem rgba(31, 33, 36, 0.18)",
    "shadow-inset-md": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
    "shadow-inset-sm": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
    "shadow-none": "none",
    "shadow-xs": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
    "shadow-sm": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
    "shadow-md": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
    "shadow-lg": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
    "shadow-xl": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
    "shadow-2xl": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
    "shadow-bevel-experimental": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
    "shadow-card-sm-experimental": "0rem 0.125rem 0rem rgba(0, 0, 0, 0.07), 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.07), 0.0625rem 0rem 0rem #E3E3E3, -0.0625rem 0rem 0rem #E3E3E3, 0rem -0.0625rem 0rem #E3E3E3",
    "shadow-card-md-experimental": "0rem 0.1875rem 0.0625rem -0.0625rem rgba(0, 0, 0, 0.07),  0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.16),  0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  -0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  0rem 0.0625rem 0rem 0rem rgba(227, 227, 227, 1)",
    "shadow-card-lg-experimental": "0rem 0.25rem 0.375rem -0.125rem rgba(0, 0, 0, 0.2), inset 0rem -0.0625rem 0rem #D4D4D4, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
    "shadow-button-experimental": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
    "shadow-button-hover-experimental": "inset 0rem -0.0625rem 0rem #cccccc, inset 0.0625rem 0rem 0rem #ebebeb, inset -0.0625rem 0rem 0rem #ebebeb, inset 0rem 0.0625rem 0rem #ebebeb",
    "shadow-button-disabled-experimental": "inset 0rem 0rem 0rem 0.0625rem rgba(227, 227, 227, 1)",
    "shadow-button-primary-strong-inset-experimental": "0rem 0.1875rem 0rem 0rem #000 inset",
    "shadow-button-primary-strong-hover-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
    "shadow-button-primary-strong-experimental": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
    "shadow-button-primary-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
    "shadow-button-primary-hover-experimental": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
    "shadow-button-inset-experimental": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
    "shadow-border-inset-experimental": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset"
  },
  "space": {
    "space-0": "0rem",
    "space-025": "0.0625rem",
    "space-050": "0.125rem",
    "space-100": "0.25rem",
    "space-150": "0.375rem",
    "space-200": "0.5rem",
    "space-300": "0.75rem",
    "space-400": "1rem",
    "space-500": "1.25rem",
    "space-600": "1.5rem",
    "space-800": "2rem",
    "space-1000": "2.5rem",
    "space-1200": "3rem",
    "space-1600": "4rem",
    "space-2000": "5rem",
    "space-2400": "6rem",
    "space-2800": "7rem",
    "space-3200": "8rem",
    "space-button-group-gap": "0.5rem",
    "space-card-gap": "1rem",
    "space-card-padding": "1rem",
    "space-table-cell-padding": "0.375rem",
    "space-05": "0.125rem",
    "space-1": "0.25rem",
    "space-1_5-experimental": "0.375rem",
    "space-2": "0.5rem",
    "space-3": "0.75rem",
    "space-4": "1rem",
    "space-5": "1.25rem",
    "space-6": "1.5rem",
    "space-8": "2rem",
    "space-10": "2.5rem",
    "space-12": "3rem",
    "space-16": "4rem",
    "space-20": "5rem",
    "space-24": "6rem",
    "space-28": "7rem",
    "space-32": "8rem"
  },
  "text": {
    "text-heading-3xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-heading-3xl-font-size": "2rem",
    "text-heading-3xl-font-weight": "700",
    "text-heading-3xl-font-letter-spacing": "-0.03375rem",
    "text-heading-3xl-font-line-height": "3rem",
    "text-heading-2xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-heading-2xl-font-size": "1.75rem",
    "text-heading-2xl-font-weight": "700",
    "text-heading-2xl-font-letter-spacing": "-0.01875rem",
    "text-heading-2xl-font-line-height": "2.5rem",
    "text-heading-xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-heading-xl-font-size": "2rem",
    "text-heading-xl-font-weight": "700",
    "text-heading-xl-font-letter-spacing": "-0.0125rem",
    "text-heading-xl-font-line-height": "2rem",
    "text-heading-lg-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-heading-lg-font-size": "1.75rem",
    "text-heading-lg-font-weight": "600",
    "text-heading-lg-font-letter-spacing": "-0.0125rem",
    "text-heading-lg-font-line-height": "1.5rem",
    "text-heading-md-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-heading-md-font-size": "0.875rem",
    "text-heading-md-font-weight": "600",
    "text-heading-md-font-letter-spacing": "0rem",
    "text-heading-md-font-line-height": "1.25rem",
    "text-heading-sm-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-heading-sm-font-size": "0.8125rem",
    "text-heading-sm-font-weight": "600",
    "text-heading-sm-font-letter-spacing": "0rem",
    "text-heading-sm-font-line-height": "1.25rem",
    "text-body-lg-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-body-lg-font-size": "0.875rem",
    "text-body-lg-font-weight": "400",
    "text-body-lg-font-letter-spacing": "0rem",
    "text-body-lg-font-line-height": "1.25rem",
    "text-body-md-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-body-md-font-size": "0.8125rem",
    "text-body-md-font-weight": "400",
    "text-body-md-font-letter-spacing": "0rem",
    "text-body-md-font-line-height": "1.25rem",
    "text-body-sm-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-body-sm-font-size": "1.25rem",
    "text-body-sm-font-weight": "400",
    "text-body-sm-font-letter-spacing": "0rem",
    "text-body-sm-font-line-height": "1rem",
    "text-body-xs-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
    "text-body-xs-font-size": "0.6875rem",
    "text-body-xs-font-weight": "400",
    "text-body-xs-font-letter-spacing": "0rem",
    "text-body-xs-font-line-height": "0.75rem"
  },
  "width": {
    "width-0": "0rem",
    "width-025": "0.0625rem",
    "width-050": "0.125rem",
    "width-100": "0.25rem",
    "width-150": "0.375rem",
    "width-200": "0.5rem",
    "width-300": "0.75rem",
    "width-400": "1rem",
    "width-500": "1.25rem",
    "width-600": "1.5rem",
    "width-700": "1.75rem",
    "width-800": "2rem",
    "width-900": "2.25rem",
    "width-1000": "2.5rem",
    "width-1200": "3rem",
    "width-1600": "4rem",
    "width-2000": "5rem",
    "width-2400": "6rem",
    "width-2800": "7rem",
    "width-3200": "8rem"
  },
  "zIndex": {
    "z-index-1": "100",
    "z-index-2": "400",
    "z-index-3": "510",
    "z-index-4": "512",
    "z-index-5": "513",
    "z-index-6": "514",
    "z-index-7": "515",
    "z-index-8": "516",
    "z-index-9": "517",
    "z-index-10": "518",
    "z-index-11": "519",
    "z-index-12": "520"
  }
};
var themes = {
  "light": {
    "border": {
      "border-radius-0": "0rem",
      "border-radius-050": "0.125rem",
      "border-radius-100": "0.25rem",
      "border-radius-150": "0.375rem",
      "border-radius-200": "0.5rem",
      "border-radius-300": "0.75rem",
      "border-radius-400": "1rem",
      "border-radius-500": "1.25rem",
      "border-radius-750": "1.875rem",
      "border-radius-full": "624.9375rem",
      "border-radius-0-experimental": "0rem",
      "border-radius-05": "0.125rem",
      "border-radius-1": "0.25rem",
      "border-radius-2": "0.5rem",
      "border-radius-3": "0.75rem",
      "border-radius-4": "1rem",
      "border-radius-5": "1.25rem",
      "border-radius-6": "1.875rem",
      "border-radius-1_5-experimental": "0.375rem",
      "border-width-0165": "0.04125rem",
      "border-width-025": "0.0625rem",
      "border-width-050": "0.125rem",
      "border-width-100": "0.25rem",
      "border-width-1": "0.0625rem",
      "border-width-2": "0.125rem",
      "border-width-3": "0.1875rem",
      "border-width-4": "0.25rem",
      "border-width-5": "0.3125rem",
      "border-width-1-experimental": "0.04125rem",
      "border-width-2-experimental": "0.0625rem"
    },
    "breakpoints": {
      "breakpoints-xs": "0rem",
      "breakpoints-sm": "30.625rem",
      "breakpoints-md": "48rem",
      "breakpoints-lg": "65rem",
      "breakpoints-xl": "90rem"
    },
    "color": {
      "color-border-inverse-active": "rgba(227, 227, 227, 1)",
      "color-border-inverse-hover": "rgba(204, 204, 204, 1)",
      "color-border-warning": "rgba(255, 200, 121, 1)",
      "color-checkbox-icon-disabled": "rgba(255, 255, 255, 1)",
      "color-checkbox-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
      "color-bg-fill-active": "rgba(247, 247, 247, 1)",
      "color-bg-fill-brand-selected": "rgba(48, 48, 48, 1)",
      "color-bg-fill-caution-active": "rgba(225, 203, 0, 1)",
      "color-bg-fill-caution-hover": "rgba(234, 211, 0, 1)",
      "color-bg-fill-critical-selected": "rgba(142, 31, 11, 1)",
      "color-bg-fill-emphasis-active": "rgba(0, 46, 106, 1)",
      "color-bg-fill-emphasis-hover": "rgba(0, 66, 153, 1)",
      "color-bg-fill-emphasis": "rgba(0, 91, 211, 1)",
      "color-bg-fill-hover": "rgba(250, 250, 250, 1)",
      "color-bg-fill-info-active": "rgba(0, 148, 213, 1)",
      "color-bg-fill-info-hover": "rgba(81, 192, 255, 1)",
      "color-bg-fill-secondary-active": "rgba(227, 227, 227, 1)",
      "color-bg-fill-secondary-hover": "rgba(235, 235, 235, 1)",
      "color-bg-fill-selected": "rgba(204, 204, 204, 1)",
      "color-bg-fill-transparent-secondary-active": "rgba(0, 0, 0, 0.11)",
      "color-bg-fill-transparent-secondary-hover": "rgba(0, 0, 0, 0.08)",
      "color-bg-fill-transparent-selected": "rgba(0, 0, 0, 0.06)",
      "color-bg-fill-transparent": "rgba(0, 0, 0, 0)",
      "color-bg-fill-warning-active": "rgba(178, 132, 0, 1)",
      "color-bg-fill-warning-hover": "rgba(229, 165, 0, 1)",
      "color-bg-fill": "rgba(255, 255, 255, 1)",
      "color-icon-secondary-active": "rgba(74, 74, 74, 1)",
      "color-icon-secondary-hover": "rgba(97, 97, 97, 1)",
      "color-text-link-active": "rgba(0, 46, 106, 1)",
      "color-text-link-hover": "rgba(0, 66, 153, 1)",
      "color-text-link": "rgba(0, 91, 211, 1)",
      "color-nav-bg": "rgba(235, 235, 235, 1)",
      "color-nav-bg-surface-active": "rgba(250, 250, 250, 1)",
      "color-nav-bg-surface-hover": "rgba(241, 241, 241, 1)",
      "color-nav-bg-surface-selected": "rgba(250, 250, 250, 1)",
      "color-nav-bg-surface": "rgba(0, 0, 0, 0.02)",
      "color-radio-button-icon-disabled": "rgba(255, 255, 255, 1)",
      "color-radio-button-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
      "color-bg-surface-emphasis-active": "rgba(226, 231, 255, 1)",
      "color-bg-surface-emphasis-hover": "rgba(234, 237, 255, 1)",
      "color-bg-surface-emphasis": "rgba(240, 242, 255, 1)",
      "color-bg-surface-inverse": "rgba(48, 48, 48, 1)",
      "color-bg-surface-magic-active": "rgba(228, 222, 255, 1)",
      "color-bg-surface-secondary-selected": "rgba(235, 235, 235, 1)",
      "color-bg-surface-tertiary-active": "rgba(227, 227, 227, 1)",
      "color-bg-surface-tertiary-hover": "rgba(235, 235, 235, 1)",
      "color-bg-surface-warning-active": "rgba(255, 228, 198, 1)",
      "color-bg-surface-warning-hover": "rgba(255, 235, 213, 1)",
      "color-text-brand-on-bg-fill-active": "rgba(204, 204, 204, 1)",
      "color-text-brand-on-bg-fill-disabled": "rgba(255, 255, 255, 1)",
      "color-text-brand-on-bg-fill-hover": "rgba(227, 227, 227, 1)",
      "color-text-brand-on-bg-fill": "rgba(255, 255, 255, 1)",
      "color-text-caution-active": "rgba(31, 28, 0, 1)",
      "color-text-caution-hover": "rgba(51, 46, 0, 1)",
      "color-text-caution-on-bg-fill": "rgba(51, 46, 0, 1)",
      "color-text-critical-on-bg-fill": "rgba(255, 251, 251, 1)",
      "color-text-emphasis-on-bg-fill-active": "rgba(213, 220, 255, 1)",
      "color-text-emphasis-on-bg-fill-hover": "rgba(226, 231, 255, 1)",
      "color-text-emphasis-on-bg-fill": "rgba(252, 253, 255, 1)",
      "color-text-info-active": "rgba(0, 33, 51, 1)",
      "color-text-info-hover": "rgba(0, 58, 90, 1)",
      "color-text-magic-on-bg-fill": "rgba(253, 253, 255, 1)",
      "color-text-success-active": "rgba(9, 42, 27, 1)",
      "color-text-success-hover": "rgba(8, 61, 37, 1)",
      "color-text-success-on-bg-fill": "rgba(248, 255, 251, 1)",
      "color-text-warning-active": "rgba(37, 26, 0, 1)",
      "color-text-warning-hover": "rgba(65, 45, 0, 1)",
      "color-text-warning-on-bg-fill": "rgba(37, 26, 0, 1)",
      "color-video-thumbnail-play-button-bg-fill-hover": "rgba(0, 0, 0, 0.81)",
      "color-video-thumbnail-play-button-bg-fill": "rgba(0, 0, 0, 0.71)",
      "color-video-thumbnail-play-button-text-on-bg-fill": "rgba(255, 255, 255, 1)",
      "color-bg-surface": "rgba(255, 255, 255, 1)",
      "color-bg-surface-hover": "rgba(247, 247, 247, 1)",
      "color-bg-surface-active": "rgba(243, 243, 243, 1)",
      "color-bg-surface-disabled": "rgba(0, 0, 0, 0.05)",
      "color-bg-surface-secondary": "rgba(249, 250, 251, 1)",
      "color-bg-surface-secondary-hover": "rgba(241, 242, 244, 1)",
      "color-bg-surface-secondary-active": "rgba(235, 236, 239, 1)",
      "color-bg-surface-tertiary": "rgba(243, 243, 243, 1)",
      "color-bg-fill-tertiary": "rgba(221, 224, 228, 1)",
      "color-bg-fill-tertiary-hover": "rgba(202, 206, 211, 1)",
      "color-bg-fill-tertiary-active": "rgba(202, 206, 211, 1)",
      "color-input-bg-surface": "rgba(255, 255, 255, 1)",
      "color-input-bg-surface-hover": "rgba(250, 250, 250, 1)",
      "color-input-bg-surface-active": "rgba(247, 247, 247, 1)",
      "color-bg-fill-brand": "rgba(48, 48, 48, 1)",
      "color-bg-fill-brand-hover": "rgba(26, 26, 26, 1)",
      "color-bg-fill-brand-active": "rgba(26, 26, 26, 1)",
      "color-bg-surface-brand": "rgba(227, 227, 227, 1)",
      "color-bg-surface-brand-hover": "rgba(235, 235, 235, 1)",
      "color-bg-surface-brand-active": "rgba(241, 241, 241, 1)",
      "color-bg-surface-brand-selected": "rgba(241, 241, 241, 1)",
      "color-bg-surface-selected": "rgba(235, 236, 239, 1)",
      "color-bg-fill-success": "rgba(22, 166, 121, 1)",
      "color-bg-fill-success-hover": "rgba(19, 111, 69, 1)",
      "color-bg-fill-success-active": "rgba(12, 81, 50, 1)",
      "color-bg-fill-success-secondary": "rgba(161, 237, 208, 1)",
      "color-bg-surface-success": "rgba(224, 248, 238, 1)",
      "color-bg-surface-success-hover": "rgba(240, 253, 248, 1)",
      "color-bg-surface-success-active": "rgba(192, 242, 221, 1)",
      "color-bg-fill-critical": "rgba(197, 40, 12, 1)",
      "color-bg-fill-critical-hover": "rgba(159, 32, 10, 1)",
      "color-bg-fill-critical-active": "rgba(115, 24, 7, 1)",
      "color-bg-fill-critical-secondary": "rgba(251, 197, 188, 1)",
      "color-bg-surface-critical": "rgba(253, 226, 221, 1)",
      "color-bg-surface-critical-hover": "rgba(254, 243, 241, 1)",
      "color-bg-surface-critical-active": "rgba(251, 197, 188, 1)",
      "color-bg-fill-caution": "rgba(216, 155, 13, 1)",
      "color-bg-fill-caution-secondary": "rgba(248, 217, 144, 1)",
      "color-bg-surface-caution": "rgba(252, 240, 212, 1)",
      "color-bg-surface-caution-hover": "rgba(254, 248, 236, 1)",
      "color-bg-surface-caution-active": "rgba(250, 229, 178, 1)",
      "color-bg-fill-info": "rgba(42, 172, 187, 1)",
      "color-bg-fill-info-secondary": "rgba(184, 233, 239, 1)",
      "color-bg-surface-info": "rgba(222, 245, 247, 1)",
      "color-bg-surface-info-hover": "rgba(238, 250, 251, 1)",
      "color-bg-surface-info-active": "rgba(184, 233, 239, 1)",
      "color-bg-fill-warning": "rgba(255, 184, 0, 1)",
      "color-bg-fill-warning-secondary": "rgba(250, 201, 168, 1)",
      "color-bg-surface-warning": "rgba(255, 241, 227, 1)",
      "color-bg-fill-magic": "rgba(121, 69, 227, 1)",
      "color-bg-fill-magic-secondary": "rgba(236, 227, 253, 1)",
      "color-bg-fill-magic-secondary-hover": "rgba(226, 214, 250, 1)",
      "color-bg-fill-magic-secondary-active": "rgba(203, 180, 248, 1)",
      "color-bg-surface-magic": "rgba(243, 241, 255, 1)",
      "color-bg-surface-magic-hover": "rgba(236, 227, 253, 1)",
      "color-bg-fill-secondary": "rgba(241, 242, 244, 1)",
      "color-bg-fill-inverse": "rgba(97, 106, 117, 1)",
      "color-bg-fill-inverse-hover": "rgba(97, 106, 117, 1)",
      "color-bg-fill-inverse-active": "rgba(135, 144, 155, 1)",
      "color-bg-surface-transparent": "rgba(0, 0, 0, 0)",
      "color-bg-fill-transparent-hover": "rgba(0, 0, 0, 0.05)",
      "color-bg-fill-transparent-active": "rgba(0, 0, 0, 0.06)",
      "color-bg-fill-disabled": "rgba(0, 0, 0, 0.05)",
      "color-bg-fill-transparent-secondary": "rgba(0, 0, 0, 0.06)",
      "color-bg-fill-brand-disabled": "rgba(0, 0, 0, 0.17)",
      "color-backdrop-bg": "rgba(0, 0, 0, 0.71)",
      "color-avatar-bg-fill": "rgba(181, 181, 181, 1)",
      "color-avatar-one-bg-fill": "rgba(251, 207, 251, 1)",
      "color-avatar-two-bg-fill": "rgba(56, 250, 163, 1)",
      "color-avatar-three-bg-fill": "rgba(146, 237, 222, 1)",
      "color-avatar-four-bg-fill": "rgba(192, 225, 255, 1)",
      "color-avatar-five-bg-fill": "rgba(254, 209, 221, 1)",
      "color-text-secondary": "rgba(97, 106, 117, 1)",
      "color-text-emphasis": "rgba(36, 99, 188, 1)",
      "color-text-emphasis-hover": "rgba(20, 73, 149, 1)",
      "color-text-emphasis-active": "rgba(14, 53, 108, 1)",
      "color-text-brand": "rgba(0, 122, 92, 1)",
      "color-text-brand-hover": "rgba(18, 84, 67, 1)",
      "color-text-critical-hover": "rgba(95, 21, 7, 1)",
      "color-text-info-on-bg-fill": "rgba(16, 65, 71, 1)",
      "color-text-warning": "rgba(94, 66, 0, 1)",
      "color-text-inverse-secondary": "rgba(171, 177, 186, 1)",
      "color-text-link-inverse": "rgba(197, 208, 255, 1)",
      "color-avatar-text-on-bg-fill": "rgba(255, 255, 255, 1)",
      "color-avatar-one-text-on-bg-fill": "rgba(121, 26, 121, 1)",
      "color-avatar-two-text-on-bg-fill": "rgba(12, 81, 50, 1)",
      "color-avatar-three-text-on-bg-fill": "rgba(9, 83, 70, 1)",
      "color-avatar-four-text-on-bg-fill": "rgba(0, 82, 124, 1)",
      "color-avatar-five-text-on-bg-fill": "rgba(141, 4, 72, 1)",
      "color-icon-secondary": "rgba(135, 144, 155, 1)",
      "color-icon-emphasis": "rgba(36, 99, 188, 1)",
      "color-icon-emphasis-hover": "rgba(20, 73, 149, 1)",
      "color-icon-emphasis-active": "rgba(14, 53, 108, 1)",
      "color-icon-brand": "rgba(0, 122, 92, 1)",
      "color-border-secondary": "rgba(221, 224, 228, 1)",
      "color-border-tertiary": "rgba(204, 204, 204, 1)",
      "color-input-border": "rgba(171, 177, 186, 1)",
      "color-input-border-hover": "rgba(97, 106, 117, 1)",
      "color-input-border-active": "rgba(26, 26, 26, 1)",
      "color-border-emphasis": "rgba(0, 91, 211, 1)",
      "color-border-emphasis-hover": "rgba(20, 73, 149, 1)",
      "color-border-emphasis-active": "rgba(14, 53, 108, 1)",
      "color-border-focus": "rgba(62, 125, 213, 1)",
      "color-border-brand": "rgba(0, 122, 92, 1)",
      "color-border-critical-secondary": "rgba(142, 31, 11, 1)",
      "color-border-magic-secondary": "rgba(121, 69, 227, 1)",
      "color-bg-inverse": "rgba(31, 33, 36, 1)",
      "color-bg-inset-strong": "rgba(97, 106, 117, 1)",
      "color-bg-inverse-hover": "rgba(97, 106, 117, 1)",
      "color-bg-inverse-active": "rgba(135, 144, 155, 1)",
      "color-bg-strong-hover": "rgba(202, 206, 211, 1)",
      "color-bg-strong-active": "rgba(202, 206, 211, 1)",
      "color-bg-strong": "rgba(221, 224, 228, 1)",
      "color-bg-subdued-active": "rgba(235, 236, 239, 1)",
      "color-bg-disabled": "rgba(235, 236, 239, 1)",
      "color-bg-interactive-disabled": "rgba(235, 236, 239, 1)",
      "color-bg-app": "rgba(241, 242, 244, 1)",
      "color-bg-app-active": "rgba(221, 224, 228, 1)",
      "color-bg-app-hover": "rgba(235, 236, 239, 1)",
      "color-bg-app-selected": "rgba(235, 236, 239, 1)",
      "color-bg-active": "rgba(235, 236, 239, 1)",
      "color-bg-subdued-hover": "rgba(241, 242, 244, 1)",
      "color-bg-inset": "rgba(241, 242, 244, 1)",
      "color-bg-hover": "rgba(241, 242, 244, 1)",
      "color-bg-subdued": "rgba(249, 250, 251, 1)",
      "color-bg-input": "rgba(255, 255, 255, 1)",
      "color-bg": "rgba(255, 255, 255, 1)",
      "color-bg-primary-active": "rgba(12, 59, 47, 1)",
      "color-bg-primary-hover": "rgba(18, 84, 67, 1)",
      "color-bg-primary": "rgba(0, 122, 92, 1)",
      "color-bg-success-strong": "rgba(22, 166, 121, 1)",
      "color-bg-success": "rgba(161, 237, 208, 1)",
      "color-bg-primary-subdued-active": "rgba(192, 242, 221, 1)",
      "color-bg-success-subdued-active": "rgba(192, 242, 221, 1)",
      "color-bg-success-subdued": "rgba(224, 248, 238, 1)",
      "color-bg-primary-subdued-hover": "rgba(224, 248, 238, 1)",
      "color-bg-success-subdued-hover": "rgba(240, 253, 248, 1)",
      "color-bg-primary-subdued": "rgba(240, 253, 248, 1)",
      "color-bg-primary-subdued-selected": "rgba(240, 253, 248, 1)",
      "color-bg-critical-strong-active": "rgba(115, 24, 7, 1)",
      "color-bg-critical-strong-hover": "rgba(159, 32, 10, 1)",
      "color-bg-critical-strong": "rgba(197, 40, 12, 1)",
      "color-bg-critical-subdued-active": "rgba(251, 197, 188, 1)",
      "color-bg-critical": "rgba(251, 197, 188, 1)",
      "color-bg-critical-subdued": "rgba(253, 226, 221, 1)",
      "color-bg-critical-subdued-hover": "rgba(254, 243, 241, 1)",
      "color-bg-caution-strong": "rgba(216, 155, 13, 1)",
      "color-bg-caution": "rgba(248, 217, 144, 1)",
      "color-bg-caution-subdued-active": "rgba(250, 229, 178, 1)",
      "color-bg-caution-subdued": "rgba(252, 240, 212, 1)",
      "color-bg-caution-subdued-hover": "rgba(254, 248, 236, 1)",
      "color-bg-info-strong": "rgba(42, 172, 187, 1)",
      "color-bg-info-subdued-active": "rgba(184, 233, 239, 1)",
      "color-bg-info": "rgba(184, 233, 239, 1)",
      "color-bg-info-subdued": "rgba(222, 245, 247, 1)",
      "color-bg-info-subdued-hover": "rgba(238, 250, 251, 1)",
      "color-bg-interactive-active": "rgba(14, 53, 108, 1)",
      "color-bg-interactive-hover": "rgba(20, 73, 149, 1)",
      "color-bg-interactive": "rgba(36, 99, 188, 1)",
      "color-bg-interactive-subdued-active": "rgba(187, 212, 247, 1)",
      "color-bg-interactive-subdued-hover": "rgba(232, 240, 253, 1)",
      "color-bg-interactive-subdued": "rgba(240, 245, 253, 1)",
      "color-bg-interactive-selected": "rgba(240, 245, 253, 1)",
      "color-bg-warning": "rgba(250, 201, 168, 1)",
      "color-bg-magic-strong": "rgba(121, 69, 227, 1)",
      "color-bg-magic-hover": "rgba(226, 214, 250, 1)",
      "color-bg-magic-active": "rgba(203, 180, 248, 1)",
      "color-bg-magic": "rgba(236, 227, 253, 1)",
      "color-bg-magic-subdued-hover": "rgba(236, 227, 253, 1)",
      "color-bg-magic-subdued-active": "rgba(226, 214, 250, 1)",
      "color-bg-magic-subdued": "rgba(242, 237, 253, 1)",
      "color-border-input-hover": "rgba(97, 106, 117, 1)",
      "color-border-inverse": "rgba(97, 106, 117, 1)",
      "color-border-strong-hover": "rgba(135, 144, 155, 1)",
      "color-border-input": "rgba(171, 177, 186, 1)",
      "color-border-hover": "rgba(171, 177, 186, 1)",
      "color-border-strong": "rgba(171, 177, 186, 1)",
      "color-border": "rgba(202, 206, 211, 1)",
      "color-border-disabled": "rgba(221, 224, 228, 1)",
      "color-border-subdued": "rgba(221, 224, 228, 1)",
      "color-border-interactive-disabled": "rgba(221, 224, 228, 1)",
      "color-border-primary": "rgba(0, 122, 92, 1)",
      "color-border-success": "rgba(22, 166, 121, 1)",
      "color-border-success-subdued": "rgba(80, 220, 169, 1)",
      "color-border-critical-active": "rgba(67, 14, 4, 1)",
      "color-border-critical-hover": "rgba(115, 24, 7, 1)",
      "color-border-critical": "rgba(197, 40, 12, 1)",
      "color-border-critical-subdued": "rgba(245, 107, 82, 1)",
      "color-border-caution": "rgba(216, 155, 13, 1)",
      "color-border-caution-subdued": "rgba(245, 196, 82, 1)",
      "color-border-info": "rgba(59, 195, 211, 1)",
      "color-border-info-subdued": "rgba(109, 211, 222, 1)",
      "color-border-interactive-active": "rgba(14, 53, 108, 1)",
      "color-border-interactive-hover": "rgba(20, 73, 149, 1)",
      "color-border-interactive": "rgba(62, 125, 213, 1)",
      "color-border-interactive-focus": "rgba(62, 125, 213, 1)",
      "color-border-interactive-subdued": "rgba(187, 212, 247, 1)",
      "color-border-magic-strong": "rgba(121, 69, 227, 1)",
      "color-border-magic": "rgba(173, 139, 241, 1)",
      "color-icon-hover": "rgba(31, 33, 36, 1)",
      "color-icon": "rgba(97, 106, 117, 1)",
      "color-icon-active": "rgba(31, 33, 36, 1)",
      "color-icon-subdued": "rgba(135, 144, 155, 1)",
      "color-icon-disabled": "rgba(171, 177, 186, 1)",
      "color-icon-interactive-disabled": "rgba(171, 177, 186, 1)",
      "color-icon-inverse": "rgba(221, 224, 228, 1)",
      "color-icon-on-color": "rgba(255, 255, 255, 1)",
      "color-icon-primary": "rgba(0, 122, 92, 1)",
      "color-icon-success": "rgba(22, 166, 121, 1)",
      "color-icon-critical": "rgba(197, 40, 12, 1)",
      "color-icon-caution": "rgba(183, 126, 11, 1)",
      "color-icon-info": "rgba(42, 172, 187, 1)",
      "color-icon-warning": "rgba(242, 117, 34, 1)",
      "color-icon-interactive-active": "rgba(14, 53, 108, 1)",
      "color-icon-interactive-hover": "rgba(20, 73, 149, 1)",
      "color-icon-interactive": "rgba(36, 99, 188, 1)",
      "color-icon-interactive-inverse": "rgba(102, 153, 225, 1)",
      "color-icon-magic": "rgba(121, 69, 227, 1)",
      "color-text": "rgba(31, 33, 36, 1)",
      "color-text-subdued": "rgba(97, 106, 117, 1)",
      "color-text-disabled": "rgba(135, 144, 155, 1)",
      "color-text-interactive-disabled": "rgba(135, 144, 155, 1)",
      "color-text-inverse-subdued": "rgba(171, 177, 186, 1)",
      "color-text-inverse": "rgba(241, 242, 244, 1)",
      "color-text-on-color": "rgba(255, 255, 255, 1)",
      "color-text-success-strong": "rgba(12, 59, 47, 1)",
      "color-text-success": "rgba(0, 122, 92, 1)",
      "color-text-primary": "rgba(0, 122, 92, 1)",
      "color-text-primary-hover": "rgba(18, 84, 67, 1)",
      "color-text-critical-strong": "rgba(67, 14, 4, 1)",
      "color-text-critical-active": "rgba(115, 24, 7, 1)",
      "color-text-critical": "rgba(197, 40, 12, 1)",
      "color-text-caution-strong": "rgba(77, 46, 5, 1)",
      "color-text-caution": "rgba(135, 92, 8, 1)",
      "color-text-info-strong": "rgba(16, 65, 71, 1)",
      "color-text-info": "rgba(32, 130, 141, 1)",
      "color-text-warning-strong": "rgba(77, 36, 5, 1)",
      "color-text-interactive-active": "rgba(14, 53, 108, 1)",
      "color-text-interactive-hover": "rgba(20, 73, 149, 1)",
      "color-text-interactive": "rgba(36, 99, 188, 1)",
      "color-text-interactive-inverse": "rgba(102, 153, 225, 1)",
      "color-text-magic-strong": "rgba(49, 13, 120, 1)",
      "color-text-magic": "rgba(90, 36, 205, 1)",
      "color-bg-backdrop-experimental": "rgba(0, 0, 0, 0.71)",
      "color-bg-primary-disabled-experimental": "rgba(212, 212, 212, 1)",
      "color-bg-secondary-experimental": "rgba(243, 243, 243, 1)",
      "color-bg-input-hover-experimental": "rgba(250, 250, 250, 1)",
      "color-border-input-active-experimental": "rgba(26, 26, 26, 1)",
      "color-border-critical-strong-experimental": "rgba(142, 31, 11, 1)",
      "color-bg-input-active-experimental": "rgba(247, 247, 247, 1)",
      "color-bg-transparent-experimental": "rgba(0, 0, 0, 0)",
      "color-bg-transparent-subdued-experimental": "rgba(0, 0, 0, 0.06)",
      "color-bg-transparent-hover-experimental": "rgba(0, 0, 0, 0.05)",
      "color-bg-transparent-active-experimental": "rgba(0, 0, 0, 0.06)",
      "color-bg-transparent-disabled-experimental": "rgba(0, 0, 0, 0.05)",
      "color-bg-transparent-secondary-disabled-experimental": "rgba(0, 0, 0, 0.08)",
      "color-bg-transparent-primary-disabled-experimental": "rgba(0, 0, 0, 0.17)",
      "color-bg-transparent-primary-experimental": "rgba(0, 0, 0, 0.62)",
      "color-bg-success-strong-hover-experimental": "rgba(19, 111, 69, 1)",
      "color-bg-success-strong-active-experimental": "rgba(12, 81, 50, 1)",
      "color-bg-warning-subdued-experimental": "rgba(255, 241, 227, 1)",
      "color-bg-warning-strong-experimental": "rgba(255, 184, 0, 1)",
      "color-text-warning-experimental": "rgba(65, 45, 0, 1)",
      "color-text-critical-hover-experimental": "rgba(95, 21, 7, 1)",
      "color-icon-info-strong-experimental": "rgba(0, 82, 124, 1)",
      "color-icon-warning-strong-experimental": "rgba(124, 88, 0, 1)",
      "color-icon-success-strong-experimental": "rgba(12, 81, 50, 1)",
      "color-icon-critical-strong-experimental": "rgba(142, 31, 11, 1)",
      "color-icon-critical-strong-hover-experimental": "rgba(95, 21, 7, 1)",
      "color-icon-critical-strong-active-experimental": "rgba(47, 10, 4, 1)",
      "color-avatar-background-experimental": "rgba(181, 181, 181, 1)",
      "color-avatar-color-experimental": "rgba(255, 255, 255, 1)",
      "color-avatar-style-one-background-experimental": "rgba(251, 207, 251, 1)",
      "color-avatar-style-one-color-experimental": "rgba(121, 26, 121, 1)",
      "color-avatar-style-two-background-experimental": "rgba(56, 250, 163, 1)",
      "color-avatar-style-two-color-experimental": "rgba(12, 81, 50, 1)",
      "color-avatar-style-three-background-experimental": "rgba(146, 237, 222, 1)",
      "color-avatar-style-three-color-experimental": "rgba(9, 83, 70, 1)",
      "color-avatar-style-four-background-experimental": "rgba(192, 225, 255, 1)",
      "color-avatar-style-four-color-experimental": "rgba(0, 82, 124, 1)",
      "color-avatar-style-five-background-experimental": "rgba(254, 209, 221, 1)",
      "color-avatar-style-five-color-experimental": "rgba(141, 4, 72, 1)"
    },
    "font": {
      "font-family-sans": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "font-family-mono": "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
      "font-size-275": "0.6875rem",
      "font-size-325": "0.8125rem",
      "font-size-350": "0.875rem",
      "font-size-750": "1.75rem",
      "font-size-900": "2rem",
      "font-size-1000": "2.5rem",
      "font-size-70-experimental": "0.6875rem",
      "font-size-75": "0.75rem",
      "font-size-80-experimental": "0.8125rem",
      "font-size-100": "0.875rem",
      "font-size-200": "1rem",
      "font-size-300": "1.25rem",
      "font-size-400": "1.5rem",
      "font-size-500": "1.75rem",
      "font-size-600": "2rem",
      "font-size-700": "2.5rem",
      "font-weight-regular": "400",
      "font-weight-medium": "500",
      "font-weight-semibold": "600",
      "font-weight-bold": "700",
      "font-letter-spacing-densest": "-0.03375rem",
      "font-letter-spacing-denser": "-0.01875rem",
      "font-letter-spacing-dense": "-0.0125rem",
      "font-letter-spacing-normal": "0rem",
      "font-line-height-300": "0.75rem",
      "font-line-height-400": "1rem",
      "font-line-height-500": "1.25rem",
      "font-line-height-600": "1.5rem",
      "font-line-height-700": "1.75rem",
      "font-line-height-800": "2rem",
      "font-line-height-1000": "2.5rem",
      "font-line-height-1200": "3rem",
      "font-line-height-075-experimental": "0.75rem",
      "font-line-height-1": "1rem",
      "font-line-height-2": "1.25rem",
      "font-line-height-3": "1.5rem",
      "font-line-height-4": "1.75rem",
      "font-line-height-5": "2rem",
      "font-line-height-6": "2.5rem",
      "font-line-height-7": "3rem"
    },
    "height": {
      "height-0": "0rem",
      "height-025": "0.0625rem",
      "height-050": "0.125rem",
      "height-100": "0.25rem",
      "height-150": "0.375rem",
      "height-200": "0.5rem",
      "height-300": "0.75rem",
      "height-400": "1rem",
      "height-500": "1.25rem",
      "height-600": "1.5rem",
      "height-700": "1.75rem",
      "height-800": "2rem",
      "height-900": "2.25rem",
      "height-1000": "2.5rem",
      "height-1200": "3rem",
      "height-1600": "4rem",
      "height-2000": "5rem",
      "height-2400": "6rem",
      "height-2800": "7rem",
      "height-3200": "8rem"
    },
    "motion": {
      "motion-duration-0": "0ms",
      "motion-duration-50": "50ms",
      "motion-duration-100": "100ms",
      "motion-duration-150": "150ms",
      "motion-duration-200": "200ms",
      "motion-duration-250": "250ms",
      "motion-duration-300": "300ms",
      "motion-duration-350": "350ms",
      "motion-duration-400": "400ms",
      "motion-duration-450": "450ms",
      "motion-duration-500": "500ms",
      "motion-duration-5000": "5000ms",
      "motion-ease": "cubic-bezier(0.25, 0.1, 0.25, 1)",
      "motion-ease-in": "cubic-bezier(0.42, 0, 1, 1)",
      "motion-ease-out": "cubic-bezier(0, 0, 0.58, 1)",
      "motion-ease-in-out": "cubic-bezier(0.42, 0, 0.58, 1)",
      "motion-linear": "cubic-bezier(0, 0, 1, 1)",
      "motion-keyframes-bounce": "{ from, 65%, 85% { transform: scale(1) } 75% { transform: scale(0.85) } 82.5% { transform: scale(1.05) } }",
      "motion-keyframes-fade-in": "{ to { opacity: 1 } }",
      "motion-keyframes-pulse": "{ from, 75% { transform: scale(0.85); opacity: 1; } to { transform: scale(2.5); opacity: 0; } }",
      "motion-keyframes-spin": "{ to { transform: rotate(1turn) } }",
      "motion-keyframes-appear-above": "{ from { transform: translateY(var(--p-space-100)); opacity: 0; } to { transform: none; opacity: 1; } }",
      "motion-keyframes-appear-below": "{ from { transform: translateY(calc(var(--p-space-100) * -1)); opacity: 0; } to { transform: none; opacity: 1; } }"
    },
    "shadow": {
      "shadow-0": "none",
      "shadow-100": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
      "shadow-200": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
      "shadow-300": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
      "shadow-400": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
      "shadow-500": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
      "shadow-600": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
      "shadow-bevel-100": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
      "shadow-inset-100": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
      "shadow-inset-200": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
      "shadow-button": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
      "shadow-button-hover": "inset 0rem -0.0625rem 0rem #CCCCCC, inset 0.0625rem 0rem 0rem #EBEBEB, inset -0.0625rem 0rem 0rem #EBEBEB, inset 0rem 0.0625rem 0rem #EBEBEB",
      "shadow-button-inset": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
      "shadow-button-primary": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
      "shadow-button-primary-hover": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
      "shadow-button-primary-inset": "0rem 0.1875rem 0rem 0rem #000 inset",
      "shadow-button-primary-critical": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
      "shadow-button-primary-critical-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
      "shadow-button-primary-critical-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
      "shadow-button-primary-success": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
      "shadow-button-primary-success-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
      "shadow-button-primary-success-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
      "shadow-border-inset": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset",
      "shadow-inset-lg": "inset 0rem 0rem 0.4375rem 0.125rem rgba(31, 33, 36, 0.18)",
      "shadow-inset-md": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
      "shadow-inset-sm": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
      "shadow-none": "none",
      "shadow-xs": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
      "shadow-sm": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
      "shadow-md": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
      "shadow-lg": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
      "shadow-xl": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
      "shadow-2xl": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
      "shadow-bevel-experimental": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
      "shadow-card-sm-experimental": "0rem 0.125rem 0rem rgba(0, 0, 0, 0.07), 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.07), 0.0625rem 0rem 0rem #E3E3E3, -0.0625rem 0rem 0rem #E3E3E3, 0rem -0.0625rem 0rem #E3E3E3",
      "shadow-card-md-experimental": "0rem 0.1875rem 0.0625rem -0.0625rem rgba(0, 0, 0, 0.07),  0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.16),  0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  -0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  0rem 0.0625rem 0rem 0rem rgba(227, 227, 227, 1)",
      "shadow-card-lg-experimental": "0rem 0.25rem 0.375rem -0.125rem rgba(0, 0, 0, 0.2), inset 0rem -0.0625rem 0rem #D4D4D4, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
      "shadow-button-experimental": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
      "shadow-button-hover-experimental": "inset 0rem -0.0625rem 0rem #cccccc, inset 0.0625rem 0rem 0rem #ebebeb, inset -0.0625rem 0rem 0rem #ebebeb, inset 0rem 0.0625rem 0rem #ebebeb",
      "shadow-button-disabled-experimental": "inset 0rem 0rem 0rem 0.0625rem rgba(227, 227, 227, 1)",
      "shadow-button-primary-strong-inset-experimental": "0rem 0.1875rem 0rem 0rem #000 inset",
      "shadow-button-primary-strong-hover-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
      "shadow-button-primary-strong-experimental": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
      "shadow-button-primary-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
      "shadow-button-primary-hover-experimental": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
      "shadow-button-inset-experimental": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
      "shadow-border-inset-experimental": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset"
    },
    "space": {
      "space-0": "0rem",
      "space-025": "0.0625rem",
      "space-050": "0.125rem",
      "space-100": "0.25rem",
      "space-150": "0.375rem",
      "space-200": "0.5rem",
      "space-300": "0.75rem",
      "space-400": "1rem",
      "space-500": "1.25rem",
      "space-600": "1.5rem",
      "space-800": "2rem",
      "space-1000": "2.5rem",
      "space-1200": "3rem",
      "space-1600": "4rem",
      "space-2000": "5rem",
      "space-2400": "6rem",
      "space-2800": "7rem",
      "space-3200": "8rem",
      "space-button-group-gap": "0.5rem",
      "space-card-gap": "1rem",
      "space-card-padding": "1rem",
      "space-table-cell-padding": "0.375rem",
      "space-05": "0.125rem",
      "space-1": "0.25rem",
      "space-1_5-experimental": "0.375rem",
      "space-2": "0.5rem",
      "space-3": "0.75rem",
      "space-4": "1rem",
      "space-5": "1.25rem",
      "space-6": "1.5rem",
      "space-8": "2rem",
      "space-10": "2.5rem",
      "space-12": "3rem",
      "space-16": "4rem",
      "space-20": "5rem",
      "space-24": "6rem",
      "space-28": "7rem",
      "space-32": "8rem"
    },
    "text": {
      "text-heading-3xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-3xl-font-size": "2rem",
      "text-heading-3xl-font-weight": "700",
      "text-heading-3xl-font-letter-spacing": "-0.03375rem",
      "text-heading-3xl-font-line-height": "3rem",
      "text-heading-2xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-2xl-font-size": "1.75rem",
      "text-heading-2xl-font-weight": "700",
      "text-heading-2xl-font-letter-spacing": "-0.01875rem",
      "text-heading-2xl-font-line-height": "2.5rem",
      "text-heading-xl-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-xl-font-size": "2rem",
      "text-heading-xl-font-weight": "700",
      "text-heading-xl-font-letter-spacing": "-0.0125rem",
      "text-heading-xl-font-line-height": "2rem",
      "text-heading-lg-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-lg-font-size": "1.75rem",
      "text-heading-lg-font-weight": "600",
      "text-heading-lg-font-letter-spacing": "-0.0125rem",
      "text-heading-lg-font-line-height": "1.5rem",
      "text-heading-md-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-md-font-size": "0.875rem",
      "text-heading-md-font-weight": "600",
      "text-heading-md-font-letter-spacing": "0rem",
      "text-heading-md-font-line-height": "1.25rem",
      "text-heading-sm-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-sm-font-size": "0.8125rem",
      "text-heading-sm-font-weight": "600",
      "text-heading-sm-font-letter-spacing": "0rem",
      "text-heading-sm-font-line-height": "1.25rem",
      "text-body-lg-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-lg-font-size": "0.875rem",
      "text-body-lg-font-weight": "400",
      "text-body-lg-font-letter-spacing": "0rem",
      "text-body-lg-font-line-height": "1.25rem",
      "text-body-md-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-md-font-size": "0.8125rem",
      "text-body-md-font-weight": "400",
      "text-body-md-font-letter-spacing": "0rem",
      "text-body-md-font-line-height": "1.25rem",
      "text-body-sm-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-sm-font-size": "1.25rem",
      "text-body-sm-font-weight": "400",
      "text-body-sm-font-letter-spacing": "0rem",
      "text-body-sm-font-line-height": "1rem",
      "text-body-xs-font-family": "-apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-xs-font-size": "0.6875rem",
      "text-body-xs-font-weight": "400",
      "text-body-xs-font-letter-spacing": "0rem",
      "text-body-xs-font-line-height": "0.75rem"
    },
    "width": {
      "width-0": "0rem",
      "width-025": "0.0625rem",
      "width-050": "0.125rem",
      "width-100": "0.25rem",
      "width-150": "0.375rem",
      "width-200": "0.5rem",
      "width-300": "0.75rem",
      "width-400": "1rem",
      "width-500": "1.25rem",
      "width-600": "1.5rem",
      "width-700": "1.75rem",
      "width-800": "2rem",
      "width-900": "2.25rem",
      "width-1000": "2.5rem",
      "width-1200": "3rem",
      "width-1600": "4rem",
      "width-2000": "5rem",
      "width-2400": "6rem",
      "width-2800": "7rem",
      "width-3200": "8rem"
    },
    "zIndex": {
      "z-index-1": "100",
      "z-index-2": "400",
      "z-index-3": "510",
      "z-index-4": "512",
      "z-index-5": "513",
      "z-index-6": "514",
      "z-index-7": "515",
      "z-index-8": "516",
      "z-index-9": "517",
      "z-index-10": "518",
      "z-index-11": "519",
      "z-index-12": "520"
    }
  },
  "Polaris-Summer-Editions-2023": {
    "border": {
      "border-radius-0": "0px",
      "border-radius-050": "2px",
      "border-radius-100": "4px",
      "border-radius-150": "6px",
      "border-radius-200": "8px",
      "border-radius-300": "12px",
      "border-radius-400": "16px",
      "border-radius-500": "20px",
      "border-radius-750": "30px",
      "border-radius-full": "624.9375rem",
      "border-radius-0-experimental": "0rem",
      "border-radius-05": "0.125rem",
      "border-radius-1": "0.25rem",
      "border-radius-2": "0.5rem",
      "border-radius-3": "0.75rem",
      "border-radius-4": "1rem",
      "border-radius-5": "1.25rem",
      "border-radius-6": "1.875rem",
      "border-radius-1_5-experimental": "0.375rem",
      "border-width-0165": "0.66px",
      "border-width-025": "1px",
      "border-width-050": "2px",
      "border-width-100": "4px",
      "border-width-1": "0.0625rem",
      "border-width-2": "0.125rem",
      "border-width-3": "0.1875rem",
      "border-width-4": "0.25rem",
      "border-width-5": "0.3125rem",
      "border-width-1-experimental": "0.04125rem",
      "border-width-2-experimental": "0.0625rem"
    },
    "breakpoints": {
      "breakpoints-xs": "0rem",
      "breakpoints-sm": "30.625rem",
      "breakpoints-md": "48rem",
      "breakpoints-lg": "65rem",
      "breakpoints-xl": "90rem"
    },
    "color": {
      "color-border-inverse-active": "rgba(227, 227, 227, 1)",
      "color-border-inverse-hover": "rgba(204, 204, 204, 1)",
      "color-border-warning": "rgba(255, 200, 121, 1)",
      "color-checkbox-icon-disabled": "rgba(255, 255, 255, 1)",
      "color-checkbox-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
      "color-bg-fill-active": "rgba(247, 247, 247, 1)",
      "color-bg-fill-brand-selected": "rgba(48, 48, 48, 1)",
      "color-bg-fill-caution-active": "rgba(225, 203, 0, 1)",
      "color-bg-fill-caution-hover": "rgba(234, 211, 0, 1)",
      "color-bg-fill-critical-selected": "rgba(142, 31, 11, 1)",
      "color-bg-fill-emphasis-active": "rgba(0, 46, 106, 1)",
      "color-bg-fill-emphasis-hover": "rgba(0, 66, 153, 1)",
      "color-bg-fill-emphasis": "rgba(0, 91, 211, 1)",
      "color-bg-fill-hover": "rgba(250, 250, 250, 1)",
      "color-bg-fill-info-active": "rgba(0, 148, 213, 1)",
      "color-bg-fill-info-hover": "rgba(81, 192, 255, 1)",
      "color-bg-fill-secondary-active": "rgba(227, 227, 227, 1)",
      "color-bg-fill-secondary-hover": "rgba(235, 235, 235, 1)",
      "color-bg-fill-selected": "rgba(204, 204, 204, 1)",
      "color-bg-fill-transparent-secondary-active": "rgba(0, 0, 0, 0.11)",
      "color-bg-fill-transparent-secondary-hover": "rgba(0, 0, 0, 0.08)",
      "color-bg-fill-transparent-selected": "rgba(0, 0, 0, 0.06)",
      "color-bg-fill-transparent": "rgba(0, 0, 0, 0)",
      "color-bg-fill-warning-active": "rgba(178, 132, 0, 1)",
      "color-bg-fill-warning-hover": "rgba(229, 165, 0, 1)",
      "color-bg-fill": "rgba(255, 255, 255, 1)",
      "color-icon-secondary-active": "rgba(74, 74, 74, 1)",
      "color-icon-secondary-hover": "rgba(97, 97, 97, 1)",
      "color-text-link-active": "rgba(0, 46, 106, 1)",
      "color-text-link-hover": "rgba(0, 66, 153, 1)",
      "color-text-link": "rgba(0, 91, 211, 1)",
      "color-nav-bg": "rgba(235, 235, 235, 1)",
      "color-nav-bg-surface-active": "rgba(250, 250, 250, 1)",
      "color-nav-bg-surface-hover": "rgba(241, 241, 241, 1)",
      "color-nav-bg-surface-selected": "rgba(250, 250, 250, 1)",
      "color-nav-bg-surface": "rgba(0, 0, 0, 0.02)",
      "color-radio-button-icon-disabled": "rgba(255, 255, 255, 1)",
      "color-radio-button-bg-surface-disabled": "rgba(0, 0, 0, 0.08)",
      "color-bg-surface-emphasis-active": "rgba(226, 231, 255, 1)",
      "color-bg-surface-emphasis-hover": "rgba(234, 237, 255, 1)",
      "color-bg-surface-emphasis": "rgba(240, 242, 255, 1)",
      "color-bg-surface-inverse": "rgba(48, 48, 48, 1)",
      "color-bg-surface-magic-active": "rgba(228, 222, 255, 1)",
      "color-bg-surface-secondary-selected": "rgba(235, 235, 235, 1)",
      "color-bg-surface-tertiary-active": "rgba(227, 227, 227, 1)",
      "color-bg-surface-tertiary-hover": "rgba(235, 235, 235, 1)",
      "color-bg-surface-warning-active": "rgba(255, 228, 198, 1)",
      "color-bg-surface-warning-hover": "rgba(255, 235, 213, 1)",
      "color-text-brand-on-bg-fill-active": "rgba(204, 204, 204, 1)",
      "color-text-brand-on-bg-fill-disabled": "rgba(255, 255, 255, 1)",
      "color-text-brand-on-bg-fill-hover": "rgba(227, 227, 227, 1)",
      "color-text-brand-on-bg-fill": "rgba(255, 255, 255, 1)",
      "color-text-caution-active": "rgba(31, 28, 0, 1)",
      "color-text-caution-hover": "rgba(51, 46, 0, 1)",
      "color-text-caution-on-bg-fill": "rgba(51, 46, 0, 1)",
      "color-text-critical-on-bg-fill": "rgba(255, 251, 251, 1)",
      "color-text-emphasis-on-bg-fill-active": "rgba(213, 220, 255, 1)",
      "color-text-emphasis-on-bg-fill-hover": "rgba(226, 231, 255, 1)",
      "color-text-emphasis-on-bg-fill": "rgba(252, 253, 255, 1)",
      "color-text-info-active": "rgba(0, 33, 51, 1)",
      "color-text-info-hover": "rgba(0, 58, 90, 1)",
      "color-text-magic-on-bg-fill": "rgba(253, 253, 255, 1)",
      "color-text-success-active": "rgba(9, 42, 27, 1)",
      "color-text-success-hover": "rgba(8, 61, 37, 1)",
      "color-text-success-on-bg-fill": "rgba(248, 255, 251, 1)",
      "color-text-warning-active": "rgba(37, 26, 0, 1)",
      "color-text-warning-hover": "rgba(65, 45, 0, 1)",
      "color-text-warning-on-bg-fill": "rgba(37, 26, 0, 1)",
      "color-video-thumbnail-play-button-bg-fill-hover": "rgba(0, 0, 0, 0.81)",
      "color-video-thumbnail-play-button-bg-fill": "rgba(0, 0, 0, 0.71)",
      "color-video-thumbnail-play-button-text-on-bg-fill": "rgba(255, 255, 255, 1)",
      "color-bg-surface": "rgba(255, 255, 255, 1)",
      "color-bg-surface-hover": "rgba(247, 247, 247, 1)",
      "color-bg-surface-active": "rgba(243, 243, 243, 1)",
      "color-bg-surface-disabled": "rgba(0, 0, 0, 0.05)",
      "color-bg-surface-secondary": "rgba(247, 247, 247, 1)",
      "color-bg-surface-secondary-hover": "rgba(241, 241, 241, 1)",
      "color-bg-surface-secondary-active": "rgba(235, 235, 235, 1)",
      "color-bg-surface-tertiary": "rgba(243, 243, 243, 1)",
      "color-bg-fill-tertiary": "rgba(227, 227, 227, 1)",
      "color-bg-fill-tertiary-hover": "rgba(212, 212, 212, 1)",
      "color-bg-fill-tertiary-active": "rgba(204, 204, 204, 1)",
      "color-input-bg-surface": "rgba(253, 253, 253, 1)",
      "color-input-bg-surface-hover": "rgba(250, 250, 250, 1)",
      "color-input-bg-surface-active": "rgba(247, 247, 247, 1)",
      "color-bg-fill-brand": "rgba(48, 48, 48, 1)",
      "color-bg-fill-brand-hover": "rgba(26, 26, 26, 1)",
      "color-bg-fill-brand-active": "rgba(26, 26, 26, 1)",
      "color-bg-surface-brand": "rgba(227, 227, 227, 1)",
      "color-bg-surface-brand-hover": "rgba(235, 235, 235, 1)",
      "color-bg-surface-brand-active": "rgba(241, 241, 241, 1)",
      "color-bg-surface-brand-selected": "rgba(241, 241, 241, 1)",
      "color-bg-surface-selected": "rgba(250, 250, 250, 1)",
      "color-bg-fill-success": "rgba(41, 132, 90, 1)",
      "color-bg-fill-success-hover": "rgba(19, 111, 69, 1)",
      "color-bg-fill-success-active": "rgba(12, 81, 50, 1)",
      "color-bg-fill-success-secondary": "rgba(205, 254, 225, 1)",
      "color-bg-surface-success": "rgba(205, 254, 225, 1)",
      "color-bg-surface-success-hover": "rgba(180, 254, 210, 1)",
      "color-bg-surface-success-active": "rgba(146, 254, 194, 1)",
      "color-bg-fill-critical": "rgba(229, 28, 0, 1)",
      "color-bg-fill-critical-hover": "rgba(181, 38, 11, 1)",
      "color-bg-fill-critical-active": "rgba(142, 31, 11, 1)",
      "color-bg-fill-critical-secondary": "rgba(254, 218, 217, 1)",
      "color-bg-surface-critical": "rgba(254, 233, 232, 1)",
      "color-bg-surface-critical-hover": "rgba(254, 226, 225, 1)",
      "color-bg-surface-critical-active": "rgba(254, 218, 217, 1)",
      "color-bg-fill-caution": "rgba(255, 230, 0, 1)",
      "color-bg-fill-caution-secondary": "rgba(255, 239, 157, 1)",
      "color-bg-surface-caution": "rgba(255, 248, 219, 1)",
      "color-bg-surface-caution-hover": "rgba(255, 244, 191, 1)",
      "color-bg-surface-caution-active": "rgba(255, 239, 157, 1)",
      "color-bg-fill-info": "rgba(145, 208, 255, 1)",
      "color-bg-fill-info-secondary": "rgba(224, 240, 255, 1)",
      "color-bg-surface-info": "rgba(234, 244, 255, 1)",
      "color-bg-surface-info-hover": "rgba(224, 240, 255, 1)",
      "color-bg-surface-info-active": "rgba(202, 230, 255, 1)",
      "color-bg-fill-warning": "rgba(255, 184, 0, 1)",
      "color-bg-fill-warning-secondary": "rgba(255, 214, 164, 1)",
      "color-bg-surface-warning": "rgba(255, 241, 227, 1)",
      "color-bg-fill-magic": "rgba(128, 81, 255, 1)",
      "color-bg-fill-magic-secondary": "rgba(228, 222, 255, 1)",
      "color-bg-fill-magic-secondary-hover": "rgba(223, 217, 255, 1)",
      "color-bg-fill-magic-secondary-active": "rgba(212, 204, 255, 1)",
      "color-bg-surface-magic": "rgba(243, 241, 255, 1)",
      "color-bg-surface-magic-hover": "rgba(239, 236, 255, 1)",
      "color-bg-fill-secondary": "rgba(241, 241, 241, 1)",
      "color-bg-fill-inverse": "rgba(48, 48, 48, 1)",
      "color-bg-fill-inverse-hover": "rgba(74, 74, 74, 1)",
      "color-bg-fill-inverse-active": "rgba(97, 97, 97, 1)",
      "color-bg-surface-transparent": "rgba(0, 0, 0, 0)",
      "color-bg-fill-transparent-hover": "rgba(0, 0, 0, 0.05)",
      "color-bg-fill-transparent-active": "rgba(0, 0, 0, 0.06)",
      "color-bg-fill-disabled": "rgba(0, 0, 0, 0.05)",
      "color-bg-fill-transparent-secondary": "rgba(0, 0, 0, 0.06)",
      "color-bg-fill-brand-disabled": "rgba(0, 0, 0, 0.17)",
      "color-backdrop-bg": "rgba(0, 0, 0, 0.71)",
      "color-avatar-bg-fill": "rgba(181, 181, 181, 1)",
      "color-avatar-one-bg-fill": "rgba(251, 207, 251, 1)",
      "color-avatar-two-bg-fill": "rgba(56, 250, 163, 1)",
      "color-avatar-three-bg-fill": "rgba(146, 237, 222, 1)",
      "color-avatar-four-bg-fill": "rgba(192, 225, 255, 1)",
      "color-avatar-five-bg-fill": "rgba(254, 209, 221, 1)",
      "color-text-secondary": "rgba(97, 97, 97, 1)",
      "color-text-emphasis": "rgba(0, 91, 211, 1)",
      "color-text-emphasis-hover": "rgba(0, 66, 153, 1)",
      "color-text-emphasis-active": "rgba(0, 46, 106, 1)",
      "color-text-brand": "rgba(74, 74, 74, 1)",
      "color-text-brand-hover": "rgba(48, 48, 48, 1)",
      "color-text-critical-hover": "rgba(95, 21, 7, 1)",
      "color-text-info-on-bg-fill": "rgba(0, 33, 51, 1)",
      "color-text-warning": "rgba(94, 66, 0, 1)",
      "color-text-inverse-secondary": "rgba(181, 181, 181, 1)",
      "color-text-link-inverse": "rgba(197, 208, 255, 1)",
      "color-avatar-text-on-bg-fill": "rgba(255, 255, 255, 1)",
      "color-avatar-one-text-on-bg-fill": "rgba(121, 26, 121, 1)",
      "color-avatar-two-text-on-bg-fill": "rgba(12, 81, 50, 1)",
      "color-avatar-three-text-on-bg-fill": "rgba(9, 83, 70, 1)",
      "color-avatar-four-text-on-bg-fill": "rgba(0, 82, 124, 1)",
      "color-avatar-five-text-on-bg-fill": "rgba(141, 4, 72, 1)",
      "color-icon-secondary": "rgba(138, 138, 138, 1)",
      "color-icon-emphasis": "rgba(0, 91, 211, 1)",
      "color-icon-emphasis-hover": "rgba(0, 66, 153, 1)",
      "color-icon-emphasis-active": "rgba(0, 46, 106, 1)",
      "color-icon-brand": "rgba(26, 26, 26, 1)",
      "color-border-secondary": "rgba(235, 235, 235, 1)",
      "color-border-tertiary": "rgba(204, 204, 204, 1)",
      "color-input-border": "rgba(138, 138, 138, 1)",
      "color-input-border-hover": "rgba(97, 97, 97, 1)",
      "color-input-border-active": "rgba(26, 26, 26, 1)",
      "color-border-emphasis": "rgba(0, 91, 211, 1)",
      "color-border-emphasis-hover": "rgba(0, 66, 153, 1)",
      "color-border-emphasis-active": "rgba(0, 46, 106, 1)",
      "color-border-focus": "rgba(0, 91, 211, 1)",
      "color-border-brand": "rgba(227, 227, 227, 1)",
      "color-border-critical-secondary": "rgba(142, 31, 11, 1)",
      "color-border-magic-secondary": "rgba(128, 81, 255, 1)",
      "color-bg-inverse": "rgba(26, 26, 26, 1)",
      "color-bg-inset-strong": "rgba(48, 48, 48, 1)",
      "color-bg-inverse-hover": "rgba(74, 74, 74, 1)",
      "color-bg-inverse-active": "rgba(97, 97, 97, 1)",
      "color-bg-strong-hover": "rgba(212, 212, 212, 1)",
      "color-bg-strong-active": "rgba(204, 204, 204, 1)",
      "color-bg-strong": "rgba(227, 227, 227, 1)",
      "color-bg-subdued-active": "rgba(235, 235, 235, 1)",
      "color-bg-disabled": "rgba(235, 235, 235, 1)",
      "color-bg-interactive-disabled": "rgba(235, 235, 235, 1)",
      "color-bg-app": "rgba(241, 241, 241, 1)",
      "color-bg-app-active": "rgba(221, 224, 228, 1)",
      "color-bg-app-hover": "rgba(253, 253, 253, 1)",
      "color-bg-app-selected": "rgba(250, 250, 250, 1)",
      "color-bg-active": "rgba(247, 247, 247, 1)",
      "color-bg-subdued-hover": "rgba(241, 241, 241, 1)",
      "color-bg-inset": "rgba(241, 241, 241, 1)",
      "color-bg-hover": "rgba(250, 250, 250, 1)",
      "color-bg-subdued": "rgba(247, 247, 247, 1)",
      "color-bg-input": "rgba(255, 255, 255, 1)",
      "color-bg": "rgba(255, 255, 255, 1)",
      "color-bg-primary-active": "rgba(26, 26, 26, 1)",
      "color-bg-primary-hover": "rgba(26, 26, 26, 1)",
      "color-bg-primary": "rgba(48, 48, 48, 1)",
      "color-bg-success-strong": "rgba(41, 132, 90, 1)",
      "color-bg-success": "rgba(205, 254, 225, 1)",
      "color-bg-primary-subdued-active": "rgba(241, 241, 241, 1)",
      "color-bg-success-subdued-active": "rgba(192, 242, 221, 1)",
      "color-bg-success-subdued": "rgba(205, 254, 225, 1)",
      "color-bg-primary-subdued-hover": "rgba(235, 235, 235, 1)",
      "color-bg-success-subdued-hover": "rgba(146, 254, 194, 1)",
      "color-bg-primary-subdued": "rgba(227, 227, 227, 1)",
      "color-bg-primary-subdued-selected": "rgba(241, 241, 241, 1)",
      "color-bg-critical-strong-active": "rgba(142, 31, 11, 1)",
      "color-bg-critical-strong-hover": "rgba(181, 38, 11, 1)",
      "color-bg-critical-strong": "rgba(229, 28, 0, 1)",
      "color-bg-critical-subdued-active": "rgba(254, 218, 217, 1)",
      "color-bg-critical": "rgba(254, 218, 217, 1)",
      "color-bg-critical-subdued": "rgba(254, 233, 232, 1)",
      "color-bg-critical-subdued-hover": "rgba(254, 226, 225, 1)",
      "color-bg-caution-strong": "rgba(255, 230, 0, 1)",
      "color-bg-caution": "rgba(255, 239, 157, 1)",
      "color-bg-caution-subdued-active": "rgba(255, 239, 157, 1)",
      "color-bg-caution-subdued": "rgba(255, 248, 219, 1)",
      "color-bg-caution-subdued-hover": "rgba(255, 244, 191, 1)",
      "color-bg-info-strong": "rgba(145, 208, 255, 1)",
      "color-bg-info-subdued-active": "rgba(202, 230, 255, 1)",
      "color-bg-info": "rgba(224, 240, 255, 1)",
      "color-bg-info-subdued": "rgba(234, 244, 255, 1)",
      "color-bg-info-subdued-hover": "rgba(224, 240, 255, 1)",
      "color-bg-interactive-active": "rgba(74, 74, 74, 1)",
      "color-bg-interactive-hover": "rgba(48, 48, 48, 1)",
      "color-bg-interactive": "rgba(26, 26, 26, 1)",
      "color-bg-interactive-subdued-active": "rgba(241, 241, 241, 1)",
      "color-bg-interactive-subdued-hover": "rgba(235, 235, 235, 1)",
      "color-bg-interactive-subdued": "rgba(227, 227, 227, 1)",
      "color-bg-interactive-selected": "rgba(241, 241, 241, 1)",
      "color-bg-warning": "rgba(255, 214, 164, 1)",
      "color-bg-magic-strong": "rgba(128, 81, 255, 1)",
      "color-bg-magic-hover": "rgba(223, 217, 255, 1)",
      "color-bg-magic-active": "rgba(212, 204, 255, 1)",
      "color-bg-magic": "rgba(228, 222, 255, 1)",
      "color-bg-magic-subdued-hover": "rgba(239, 236, 255, 1)",
      "color-bg-magic-subdued-active": "rgba(228, 222, 255, 1)",
      "color-bg-magic-subdued": "rgba(243, 241, 255, 1)",
      "color-border-input-hover": "rgba(97, 97, 97, 1)",
      "color-border-inverse": "rgba(97, 97, 97, 1)",
      "color-border-strong-hover": "rgba(181, 181, 181, 1)",
      "color-border-input": "rgba(138, 138, 138, 1)",
      "color-border-hover": "rgba(204, 204, 204, 1)",
      "color-border-strong": "rgba(204, 204, 204, 1)",
      "color-border": "rgba(227, 227, 227, 1)",
      "color-border-disabled": "rgba(235, 235, 235, 1)",
      "color-border-subdued": "rgba(235, 235, 235, 1)",
      "color-border-interactive-disabled": "rgba(235, 235, 235, 1)",
      "color-border-primary": "rgba(227, 227, 227, 1)",
      "color-border-success": "rgba(146, 254, 194, 1)",
      "color-border-success-subdued": "rgba(146, 254, 194, 1)",
      "color-border-critical-active": "rgba(239, 77, 47, 1)",
      "color-border-critical-hover": "rgba(253, 129, 122, 1)",
      "color-border-critical": "rgba(253, 176, 172, 1)",
      "color-border-critical-subdued": "rgba(253, 176, 172, 1)",
      "color-border-caution": "rgba(255, 235, 120, 1)",
      "color-border-caution-subdued": "rgba(255, 235, 120, 1)",
      "color-border-info": "rgba(145, 208, 255, 1)",
      "color-border-info-subdued": "rgba(145, 208, 255, 1)",
      "color-border-interactive-active": "rgba(0, 46, 106, 1)",
      "color-border-interactive-hover": "rgba(0, 66, 153, 1)",
      "color-border-interactive": "rgba(0, 91, 211, 1)",
      "color-border-interactive-focus": "rgba(0, 91, 211, 1)",
      "color-border-interactive-subdued": "rgba(0, 91, 211, 1)",
      "color-border-magic-strong": "rgba(128, 81, 255, 1)",
      "color-border-magic": "rgba(170, 149, 255, 1)",
      "color-icon-hover": "rgba(48, 48, 48, 1)",
      "color-icon": "rgba(74, 74, 74, 1)",
      "color-icon-active": "rgba(31, 33, 36, 1)",
      "color-icon-subdued": "rgba(138, 138, 138, 1)",
      "color-icon-disabled": "rgba(204, 204, 204, 1)",
      "color-icon-interactive-disabled": "rgba(204, 204, 204, 1)",
      "color-icon-inverse": "rgba(227, 227, 227, 1)",
      "color-icon-on-color": "rgba(255, 255, 255, 1)",
      "color-icon-primary": "rgba(26, 26, 26, 1)",
      "color-icon-success": "rgba(41, 132, 90, 1)",
      "color-icon-critical": "rgba(239, 77, 47, 1)",
      "color-icon-caution": "rgba(153, 138, 0, 1)",
      "color-icon-info": "rgba(0, 148, 213, 1)",
      "color-icon-warning": "rgba(178, 132, 0, 1)",
      "color-icon-interactive-active": "rgba(0, 46, 106, 1)",
      "color-icon-interactive-hover": "rgba(0, 66, 153, 1)",
      "color-icon-interactive": "rgba(0, 91, 211, 1)",
      "color-icon-interactive-inverse": "rgba(197, 208, 255, 1)",
      "color-icon-magic": "rgba(113, 38, 255, 1)",
      "color-text": "rgba(48, 48, 48, 1)",
      "color-text-subdued": "rgba(97, 97, 97, 1)",
      "color-text-disabled": "rgba(181, 181, 181, 1)",
      "color-text-interactive-disabled": "rgba(204, 204, 204, 1)",
      "color-text-inverse-subdued": "rgba(204, 204, 204, 1)",
      "color-text-inverse": "rgba(227, 227, 227, 1)",
      "color-text-on-color": "rgba(255, 255, 255, 1)",
      "color-text-success-strong": "rgba(8, 61, 37, 1)",
      "color-text-success": "rgba(8, 61, 37, 1)",
      "color-text-primary": "rgba(74, 74, 74, 1)",
      "color-text-primary-hover": "rgba(74, 74, 74, 1)",
      "color-text-critical-strong": "rgba(142, 31, 11, 1)",
      "color-text-critical-active": "rgba(47, 10, 4, 1)",
      "color-text-critical": "rgba(142, 31, 11, 1)",
      "color-text-caution-strong": "rgba(51, 46, 0, 1)",
      "color-text-caution": "rgba(51, 46, 0, 1)",
      "color-text-info-strong": "rgba(0, 33, 51, 1)",
      "color-text-info": "rgba(0, 82, 124, 1)",
      "color-text-warning-strong": "rgba(37, 26, 0, 1)",
      "color-text-interactive-active": "rgba(0, 46, 106, 1)",
      "color-text-interactive-hover": "rgba(0, 66, 153, 1)",
      "color-text-interactive": "rgba(0, 91, 211, 1)",
      "color-text-interactive-inverse": "rgba(197, 208, 255, 1)",
      "color-text-magic-strong": "rgba(59, 0, 147, 1)",
      "color-text-magic": "rgba(87, 0, 209, 1)",
      "color-bg-backdrop-experimental": "rgba(0, 0, 0, 0.71)",
      "color-bg-primary-disabled-experimental": "rgba(212, 212, 212, 1)",
      "color-bg-secondary-experimental": "rgba(243, 243, 243, 1)",
      "color-bg-input-hover-experimental": "rgba(250, 250, 250, 1)",
      "color-border-input-active-experimental": "rgba(26, 26, 26, 1)",
      "color-border-critical-strong-experimental": "rgba(142, 31, 11, 1)",
      "color-bg-input-active-experimental": "rgba(247, 247, 247, 1)",
      "color-bg-transparent-experimental": "rgba(0, 0, 0, 0)",
      "color-bg-transparent-subdued-experimental": "rgba(0, 0, 0, 0.06)",
      "color-bg-transparent-hover-experimental": "rgba(0, 0, 0, 0.05)",
      "color-bg-transparent-active-experimental": "rgba(0, 0, 0, 0.06)",
      "color-bg-transparent-disabled-experimental": "rgba(0, 0, 0, 0.05)",
      "color-bg-transparent-secondary-disabled-experimental": "rgba(0, 0, 0, 0.08)",
      "color-bg-transparent-primary-disabled-experimental": "rgba(0, 0, 0, 0.17)",
      "color-bg-transparent-primary-experimental": "rgba(0, 0, 0, 0.62)",
      "color-bg-success-strong-hover-experimental": "rgba(19, 111, 69, 1)",
      "color-bg-success-strong-active-experimental": "rgba(12, 81, 50, 1)",
      "color-bg-warning-subdued-experimental": "rgba(255, 241, 227, 1)",
      "color-bg-warning-strong-experimental": "rgba(255, 184, 0, 1)",
      "color-text-warning-experimental": "rgba(65, 45, 0, 1)",
      "color-text-critical-hover-experimental": "rgba(95, 21, 7, 1)",
      "color-icon-info-strong-experimental": "rgba(0, 82, 124, 1)",
      "color-icon-warning-strong-experimental": "rgba(124, 88, 0, 1)",
      "color-icon-success-strong-experimental": "rgba(12, 81, 50, 1)",
      "color-icon-critical-strong-experimental": "rgba(142, 31, 11, 1)",
      "color-icon-critical-strong-hover-experimental": "rgba(95, 21, 7, 1)",
      "color-icon-critical-strong-active-experimental": "rgba(47, 10, 4, 1)",
      "color-avatar-background-experimental": "rgba(181, 181, 181, 1)",
      "color-avatar-color-experimental": "rgba(255, 255, 255, 1)",
      "color-avatar-style-one-background-experimental": "rgba(251, 207, 251, 1)",
      "color-avatar-style-one-color-experimental": "rgba(121, 26, 121, 1)",
      "color-avatar-style-two-background-experimental": "rgba(56, 250, 163, 1)",
      "color-avatar-style-two-color-experimental": "rgba(12, 81, 50, 1)",
      "color-avatar-style-three-background-experimental": "rgba(146, 237, 222, 1)",
      "color-avatar-style-three-color-experimental": "rgba(9, 83, 70, 1)",
      "color-avatar-style-four-background-experimental": "rgba(192, 225, 255, 1)",
      "color-avatar-style-four-color-experimental": "rgba(0, 82, 124, 1)",
      "color-avatar-style-five-background-experimental": "rgba(254, 209, 221, 1)",
      "color-avatar-style-five-color-experimental": "rgba(141, 4, 72, 1)"
    },
    "font": {
      "font-family-sans": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "font-family-mono": "ui-monospace, SFMono-Regular, 'SF Mono', Consolas, 'Liberation Mono', Menlo, monospace",
      "font-size-275": "11px",
      "font-size-325": "13px",
      "font-size-350": "14px",
      "font-size-750": "30px",
      "font-size-900": "36px",
      "font-size-1000": "40px",
      "font-size-70-experimental": "0.6875rem",
      "font-size-75": "0.75rem",
      "font-size-80-experimental": "0.8125rem",
      "font-size-100": "0.875rem",
      "font-size-200": "1rem",
      "font-size-300": "1.25rem",
      "font-size-400": "1.5rem",
      "font-size-500": "30px",
      "font-size-600": "36px",
      "font-size-700": "2.5rem",
      "font-weight-regular": "400",
      "font-weight-medium": "500",
      "font-weight-semibold": "600",
      "font-weight-bold": "700",
      "font-letter-spacing-densest": "-0.03375rem",
      "font-letter-spacing-denser": "-0.01875rem",
      "font-letter-spacing-dense": "-0.0125rem",
      "font-letter-spacing-normal": "0rem",
      "font-line-height-300": "12px",
      "font-line-height-400": "16px",
      "font-line-height-500": "20px",
      "font-line-height-600": "24px",
      "font-line-height-700": "28px",
      "font-line-height-800": "32px",
      "font-line-height-1000": "40px",
      "font-line-height-1200": "48px",
      "font-line-height-075-experimental": "0.75rem",
      "font-line-height-1": "1rem",
      "font-line-height-2": "1.25rem",
      "font-line-height-3": "1.5rem",
      "font-line-height-4": "1.75rem",
      "font-line-height-5": "2rem",
      "font-line-height-6": "2.5rem",
      "font-line-height-7": "3rem"
    },
    "height": {
      "height-0": "0rem",
      "height-025": "0.0625rem",
      "height-050": "0.125rem",
      "height-100": "0.25rem",
      "height-150": "0.375rem",
      "height-200": "0.5rem",
      "height-300": "0.75rem",
      "height-400": "1rem",
      "height-500": "1.25rem",
      "height-600": "1.5rem",
      "height-700": "1.75rem",
      "height-800": "2rem",
      "height-900": "2.25rem",
      "height-1000": "2.5rem",
      "height-1200": "3rem",
      "height-1600": "4rem",
      "height-2000": "5rem",
      "height-2400": "6rem",
      "height-2800": "7rem",
      "height-3200": "8rem"
    },
    "motion": {
      "motion-duration-0": "0ms",
      "motion-duration-50": "50ms",
      "motion-duration-100": "100ms",
      "motion-duration-150": "150ms",
      "motion-duration-200": "200ms",
      "motion-duration-250": "250ms",
      "motion-duration-300": "300ms",
      "motion-duration-350": "350ms",
      "motion-duration-400": "400ms",
      "motion-duration-450": "450ms",
      "motion-duration-500": "500ms",
      "motion-duration-5000": "5000ms",
      "motion-ease": "cubic-bezier(0.25, 0.1, 0.25, 1)",
      "motion-ease-in": "cubic-bezier(0.42, 0, 1, 1)",
      "motion-ease-out": "cubic-bezier(0.19, 0.91, 0.38, 1)",
      "motion-ease-in-out": "cubic-bezier(0.42, 0, 0.58, 1)",
      "motion-linear": "cubic-bezier(0, 0, 1, 1)",
      "motion-keyframes-bounce": "{ from, 65%, 85% { transform: scale(1) } 75% { transform: scale(0.85) } 82.5% { transform: scale(1.05) } }",
      "motion-keyframes-fade-in": "{ to { opacity: 1 } }",
      "motion-keyframes-pulse": "{ from, 75% { transform: scale(0.85); opacity: 1; } to { transform: scale(2.5); opacity: 0; } }",
      "motion-keyframes-spin": "{ to { transform: rotate(1turn) } }",
      "motion-keyframes-appear-above": "{ from { transform: translateY(var(--p-space-100)); opacity: 0; } to { transform: none; opacity: 1; } }",
      "motion-keyframes-appear-below": "{ from { transform: translateY(calc(var(--p-space-100) * -1)); opacity: 0; } to { transform: none; opacity: 1; } }"
    },
    "shadow": {
      "shadow-0": "none",
      "shadow-100": "0rem 0rem 0.125rem rgba(31, 33, 36, 0.24)",
      "shadow-200": "0rem 0.0625rem 0.0625rem rgba(31, 33, 36, 0.1)",
      "shadow-300": "0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.1), 0rem 0.0625rem 0.375rem rgba(31, 33, 36, 0.05)",
      "shadow-400": "0rem 0.25rem 0.75rem rgba(31, 33, 36, 0.2), 0rem 0.125rem 0.375rem rgba(31, 33, 36, 0.05)",
      "shadow-500": "0rem 0.25rem 1.125rem -0.125rem rgba(31, 33, 36, 0.08), 0rem 0.75rem 1.125rem -0.125rem rgba(31, 33, 36, 0.15)",
      "shadow-600": "0rem 2rem 2rem rgba(31, 33, 36, 0.15), 0rem 2rem 3.5rem -0.125rem rgba(31, 33, 36, 0.16)",
      "shadow-bevel-100": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
      "shadow-inset-100": "inset 0rem 0rem 0.1875rem rgba(31, 33, 36, 0.56)",
      "shadow-inset-200": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
      "shadow-button": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
      "shadow-button-hover": "inset 0rem -0.0625rem 0rem #CCCCCC, inset 0.0625rem 0rem 0rem #EBEBEB, inset -0.0625rem 0rem 0rem #EBEBEB, inset 0rem 0.0625rem 0rem #EBEBEB",
      "shadow-button-inset": "inset 0rem 0.125rem 0.25rem rgba(31, 33, 36, 0.32)",
      "shadow-button-primary": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
      "shadow-button-primary-hover": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
      "shadow-button-primary-inset": "0rem 0.1875rem 0rem 0rem #000 inset",
      "shadow-button-primary-critical": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
      "shadow-button-primary-critical-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
      "shadow-button-primary-critical-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
      "shadow-button-primary-success": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
      "shadow-button-primary-success-hover": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
      "shadow-button-primary-success-inset": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
      "shadow-border-inset": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset",
      "shadow-inset-lg": "inset -1px 0px 1px rgba(0, 0, 0, 0.2), inset 1px 0px 1px rgba(0, 0, 0, 0.2), inset 0px 2px 1px rgba(0, 0, 0, 0.6)",
      "shadow-inset-md": "inset -1px 0px 1px rgba(0, 0, 0, 0.12), inset 1px 0px 1px rgba(0, 0, 0, 0.12), inset 0px 2px 1px rgba(0, 0, 0, 0.2)",
      "shadow-inset-sm": "inset 0px 1px 1px rgba(0, 0, 0, 0.15), inset 0px 1px 2px rgba(255, 255, 255, 0.15)",
      "shadow-none": "none",
      "shadow-xs": "0px 1px 0px rgba(0, 0, 0, 0.07)",
      "shadow-sm": "0px 3px 1px -1px rgba(0, 0, 0, 0.07)",
      "shadow-md": "0px 4px 6px -2px rgba(0, 0, 0, 0.2)",
      "shadow-lg": "0px 8px 16px -4px rgba(0, 0, 0, 0.22)",
      "shadow-xl": "0px 12px 20px -8px rgba(0, 0, 0, 0.24)",
      "shadow-2xl": "0px 20px 20px -8px rgba(0, 0, 0, 0.28)",
      "shadow-bevel-experimental": "0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, -0.0625rem 0rem 0rem 0rem rgba(0, 0, 0, 0.13) inset, 0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.17) inset, 0rem 0.0625rem 0rem 0rem rgba(204, 204, 204, 0.5) inset",
      "shadow-card-sm-experimental": "0rem 0.125rem 0rem rgba(0, 0, 0, 0.07), 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.07), 0.0625rem 0rem 0rem #E3E3E3, -0.0625rem 0rem 0rem #E3E3E3, 0rem -0.0625rem 0rem #E3E3E3",
      "shadow-card-md-experimental": "0rem 0.1875rem 0.0625rem -0.0625rem rgba(0, 0, 0, 0.07),  0rem -0.0625rem 0rem 0rem rgba(0, 0, 0, 0.16),  0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  -0.0625rem 0rem 0rem 0rem rgba(227, 227, 227, 1),  0rem 0.0625rem 0rem 0rem rgba(227, 227, 227, 1)",
      "shadow-card-lg-experimental": "0rem 0.25rem 0.375rem -0.125rem rgba(0, 0, 0, 0.2), inset 0rem -0.0625rem 0rem #D4D4D4, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
      "shadow-button-experimental": "inset 0rem -0.0625rem 0rem #B5B5B5, inset -0.0625rem 0rem 0rem #E3E3E3, inset 0.0625rem 0rem 0rem #E3E3E3, inset 0rem 0.0625rem 0rem #E3E3E3",
      "shadow-button-hover-experimental": "inset 0rem -0.0625rem 0rem #cccccc, inset 0.0625rem 0rem 0rem #ebebeb, inset -0.0625rem 0rem 0rem #ebebeb, inset 0rem 0.0625rem 0rem #ebebeb",
      "shadow-button-disabled-experimental": "inset 0rem 0rem 0rem 0.0625rem rgba(227, 227, 227, 1)",
      "shadow-button-primary-strong-inset-experimental": "0rem 0.1875rem 0rem 0rem #000 inset",
      "shadow-button-primary-strong-hover-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.16) inset, 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0rem 0rem 0.03125rem #1A1A1A",
      "shadow-button-primary-strong-experimental": "0rem 0.125rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.125rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.0625rem 0rem 0.0625rem #000 inset, 0rem 0.0625rem 0rem 0rem #000 inset",
      "shadow-button-primary-experimental": "0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.4) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, -0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.2) inset, 0rem -0.09375rem 0rem 0rem rgba(0, 0, 0, 0.25) inset",
      "shadow-button-primary-hover-experimental": "-0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0.0625rem 0rem 0rem 0rem rgba(255, 255, 255, 0.20) inset, 0rem 0.03125rem 0rem 0rem rgba(0, 0, 0, 0.25), 0rem -0.09375rem 0rem 0rem rgba(255, 255, 255, 0.07) inset, 0rem 0.0625rem 0rem 0rem rgba(255, 255, 255, 0.24) inset",
      "shadow-button-inset-experimental": "0rem 0.125rem 0rem 0rem rgba(0, 0, 0, 0.60) inset, 0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset, -0.0625rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.20) inset",
      "shadow-border-inset-experimental": "0rem 0rem 0rem 0.0625rem rgba(0, 0, 0, 0.08) inset"
    },
    "space": {
      "space-0": "0rem",
      "space-025": "0.0625rem",
      "space-050": "2px",
      "space-100": "4px",
      "space-150": "6px",
      "space-200": "8px",
      "space-300": "12px",
      "space-400": "16px",
      "space-500": "20px",
      "space-600": "24px",
      "space-800": "32px",
      "space-1000": "40px",
      "space-1200": "48px",
      "space-1600": "64px",
      "space-2000": "80px",
      "space-2400": "96px",
      "space-2800": "112px",
      "space-3200": "128px",
      "space-button-group-gap": "8px",
      "space-card-gap": "16px",
      "space-card-padding": "16px",
      "space-table-cell-padding": "6px",
      "space-05": "0.125rem",
      "space-1": "0.25rem",
      "space-1_5-experimental": "0.375rem",
      "space-2": "0.5rem",
      "space-3": "0.75rem",
      "space-4": "1rem",
      "space-5": "1.25rem",
      "space-6": "1.5rem",
      "space-8": "2rem",
      "space-10": "2.5rem",
      "space-12": "3rem",
      "space-16": "4rem",
      "space-20": "5rem",
      "space-24": "6rem",
      "space-28": "7rem",
      "space-32": "8rem"
    },
    "text": {
      "text-heading-3xl-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-3xl-font-size": "36px",
      "text-heading-3xl-font-weight": "700",
      "text-heading-3xl-font-letter-spacing": "-0.03375rem",
      "text-heading-3xl-font-line-height": "48px",
      "text-heading-2xl-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-2xl-font-size": "30px",
      "text-heading-2xl-font-weight": "700",
      "text-heading-2xl-font-letter-spacing": "-0.01875rem",
      "text-heading-2xl-font-line-height": "40px",
      "text-heading-xl-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-xl-font-size": "36px",
      "text-heading-xl-font-weight": "700",
      "text-heading-xl-font-letter-spacing": "-0.0125rem",
      "text-heading-xl-font-line-height": "32px",
      "text-heading-lg-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-lg-font-size": "30px",
      "text-heading-lg-font-weight": "600",
      "text-heading-lg-font-letter-spacing": "-0.0125rem",
      "text-heading-lg-font-line-height": "24px",
      "text-heading-md-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-md-font-size": "14px",
      "text-heading-md-font-weight": "600",
      "text-heading-md-font-letter-spacing": "0rem",
      "text-heading-md-font-line-height": "20px",
      "text-heading-sm-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-heading-sm-font-size": "13px",
      "text-heading-sm-font-weight": "600",
      "text-heading-sm-font-letter-spacing": "0rem",
      "text-heading-sm-font-line-height": "20px",
      "text-body-lg-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-lg-font-size": "14px",
      "text-body-lg-font-weight": "400",
      "text-body-lg-font-letter-spacing": "0rem",
      "text-body-lg-font-line-height": "20px",
      "text-body-md-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-md-font-size": "13px",
      "text-body-md-font-weight": "400",
      "text-body-md-font-letter-spacing": "0rem",
      "text-body-md-font-line-height": "20px",
      "text-body-sm-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-sm-font-size": "1.25rem",
      "text-body-sm-font-weight": "400",
      "text-body-sm-font-letter-spacing": "0rem",
      "text-body-sm-font-line-height": "16px",
      "text-body-xs-font-family": "'Inter', -apple-system, BlinkMacSystemFont, 'San Francisco', 'Segoe UI', Roboto, 'Helvetica Neue', sans-serif",
      "text-body-xs-font-size": "11px",
      "text-body-xs-font-weight": "400",
      "text-body-xs-font-letter-spacing": "0rem",
      "text-body-xs-font-line-height": "12px"
    },
    "width": {
      "width-0": "0rem",
      "width-025": "0.0625rem",
      "width-050": "0.125rem",
      "width-100": "0.25rem",
      "width-150": "0.375rem",
      "width-200": "0.5rem",
      "width-300": "0.75rem",
      "width-400": "1rem",
      "width-500": "1.25rem",
      "width-600": "1.5rem",
      "width-700": "1.75rem",
      "width-800": "2rem",
      "width-900": "2.25rem",
      "width-1000": "2.5rem",
      "width-1200": "3rem",
      "width-1600": "4rem",
      "width-2000": "5rem",
      "width-2400": "6rem",
      "width-2800": "7rem",
      "width-3200": "8rem"
    },
    "zIndex": {
      "z-index-1": "100",
      "z-index-2": "400",
      "z-index-3": "510",
      "z-index-4": "512",
      "z-index-5": "513",
      "z-index-6": "514",
      "z-index-7": "515",
      "z-index-8": "516",
      "z-index-9": "517",
      "z-index-10": "518",
      "z-index-11": "519",
      "z-index-12": "520"
    }
  }
};

export { border, breakpoints, color, font, height, motion, shadow, space, text, themes, tokens, width, zIndex };
