// @mdxeditor/editor/dist/styles/ui.module.css.js
const editorRoot = "_editorRoot_lug8m_37";
const editorWrapper = "_editorWrapper_lug8m_138";
const nestedListItem = "_nestedListItem_lug8m_141";
const toolbarRoot = "_toolbarRoot_lug8m_145";
const readOnlyToolbarRoot = "_readOnlyToolbarRoot_lug8m_168";
const toolbarModeSwitch = "_toolbarModeSwitch_lug8m_176";
const toolbarToggleItem = "_toolbarToggleItem_lug8m_186";
const toolbarGroupOfGroups = "_toolbarGroupOfGroups_lug8m_196";
const toolbarToggleSingleGroup = "_toolbarToggleSingleGroup_lug8m_201";
const toolbarButton = "_toolbarButton_lug8m_215";
const activeToolbarButton = "_activeToolbarButton_lug8m_246";
const toolbarNodeKindSelectContainer = "_toolbarNodeKindSelectContainer_lug8m_254";
const toolbarButtonDropdownContainer = "_toolbarButtonDropdownContainer_lug8m_255";
const toolbarCodeBlockLanguageSelectContent = "_toolbarCodeBlockLanguageSelectContent_lug8m_256";
const selectContainer = "_selectContainer_lug8m_257";
const selectItem = "_selectItem_lug8m_271";
const toolbarNodeKindSelectTrigger = "_toolbarNodeKindSelectTrigger_lug8m_276";
const toolbarButtonSelectTrigger = "_toolbarButtonSelectTrigger_lug8m_277";
const selectTrigger = "_selectTrigger_lug8m_278";
const toolbarCodeBlockLanguageSelectTrigger = "_toolbarCodeBlockLanguageSelectTrigger_lug8m_312";
const toolbarNodeKindSelectItem = "_toolbarNodeKindSelectItem_lug8m_318";
const toolbarNodeKindSelectDropdownArrow = "_toolbarNodeKindSelectDropdownArrow_lug8m_344";
const selectDropdownArrow = "_selectDropdownArrow_lug8m_345";
const contentEditable = "_contentEditable_lug8m_351";
const sandpackWrapper = "_sandpackWrapper_lug8m_361";
const frontmatterWrapper = "_frontmatterWrapper_lug8m_365";
const frontmatterToggleButton = "_frontmatterToggleButton_lug8m_375";
const propertyPanelTitle = "_propertyPanelTitle_lug8m_383";
const propertyEditorTable = "_propertyEditorTable_lug8m_391";
const iconButton = "_iconButton_lug8m_408";
const readOnlyColumnCell = "_readOnlyColumnCell_lug8m_413";
const propertyEditorLabelCell = "_propertyEditorLabelCell_lug8m_418";
const buttonsFooter = "_buttonsFooter_lug8m_426";
const propertyEditorInput = "_propertyEditorInput_lug8m_432";
const primaryButton = "_primaryButton_lug8m_453";
const secondaryButton = "_secondaryButton_lug8m_453";
const smallButton = "_smallButton_lug8m_467";
const dialogForm = "_dialogForm_lug8m_479";
const linkDialogEditForm = "_linkDialogEditForm_lug8m_485";
const linkDialogInputContainer = "_linkDialogInputContainer_lug8m_493";
const linkDialogInputWrapper = "_linkDialogInputWrapper_lug8m_499";
const linkDialogInput = "_linkDialogInput_lug8m_493";
const dialogInput = "_dialogInput_lug8m_519";
const linkDialogAnchor = "_linkDialogAnchor_lug8m_529";
const linkDialogPopoverContent = "_linkDialogPopoverContent_lug8m_543";
const tableColumnEditorPopoverContent = "_tableColumnEditorPopoverContent_lug8m_543";
const dialogContent = "_dialogContent_lug8m_543";
const largeDialogContent = "_largeDialogContent_lug8m_555";
const dialogTitle = "_dialogTitle_lug8m_565";
const dialogCloseButton = "_dialogCloseButton_lug8m_571";
const popoverContent = "_popoverContent_lug8m_578";
const popoverArrow = "_popoverArrow_lug8m_590";
const linkDialogPreviewAnchor = "_linkDialogPreviewAnchor_lug8m_594";
const tooltipTrigger = "_tooltipTrigger_lug8m_615";
const tooltipContent = "_tooltipContent_lug8m_619";
const actionButton = "_actionButton_lug8m_633";
const primaryActionButton = "_primaryActionButton_lug8m_639";
const tableEditor = "_tableEditor_lug8m_648";
const toolCell = "_toolCell_lug8m_659";
const tableColumnEditorTrigger = "_tableColumnEditorTrigger_lug8m_676";
const tableRowEditorTrigger = "_tableRowEditorTrigger_lug8m_676";
const addRowButton = "_addRowButton_lug8m_676";
const addColumnButton = "_addColumnButton_lug8m_676";
const tableColumnEditorToolbar = "_tableColumnEditorToolbar_lug8m_710";
const toggleGroupRoot = "_toggleGroupRoot_lug8m_723";
const tableToolsColumn = "_tableToolsColumn_lug8m_740";
const leftAlignedCell = "_leftAlignedCell_lug8m_748";
const rightAlignedCell = "_rightAlignedCell_lug8m_752";
const centeredCell = "_centeredCell_lug8m_756";
const dialogOverlay = "_dialogOverlay_lug8m_787";
const overlayShow = "_overlayShow_lug8m_1";
const contentShow = "_contentShow_lug8m_1";
const focusedImage = "_focusedImage_lug8m_830";
const imageWrapper = "_imageWrapper_lug8m_834";
const editImageButton = "_editImageButton_lug8m_846";
const inlineEditor = "_inlineEditor_lug8m_858";
const blockEditor = "_blockEditor_lug8m_867";
const nestedEditor = "_nestedEditor_lug8m_876";
const genericComponentName = "_genericComponentName_lug8m_893";
const diffSourceToggle = "_diffSourceToggle_lug8m_899";
const selectWithLabel = "_selectWithLabel_lug8m_912";
const toolbarTitleMode = "_toolbarTitleMode_lug8m_926";
const imageControlWrapperResizing = "_imageControlWrapperResizing_lug8m_932";
const imageResizer = "_imageResizer_lug8m_936";
const imageResizerN = "_imageResizerN_lug8m_945";
const imageResizerNe = "_imageResizerNe_lug8m_951";
const imageResizerE = "_imageResizerE_lug8m_957";
const imageResizerSe = "_imageResizerSe_lug8m_963";
const imageResizerS = "_imageResizerS_lug8m_963";
const imageResizerSw = "_imageResizerSw_lug8m_975";
const imageResizerW = "_imageResizerW_lug8m_981";
const imageResizerNw = "_imageResizerNw_lug8m_987";
const placeholder = "_placeholder_lug8m_993";
const rootContentEditableWrapper = "_rootContentEditableWrapper_lug8m_1006";
const downshiftContainer = "_downshiftContainer_lug8m_1012";
const downshiftInputWrapper = "_downshiftInputWrapper_lug8m_1018";
const downshiftInput = "_downshiftInput_lug8m_1018";
const downshiftAutocompleteContainer = "_downshiftAutocompleteContainer_lug8m_1048";
const textInput = "_textInput_lug8m_1093";
const multiFieldForm = "_multiFieldForm_lug8m_1101";
const formField = "_formField_lug8m_1107";
const markdownParseError = "_markdownParseError_lug8m_1118";
const popupContainer = "_popupContainer_lug8m_1127";
const styles = {
  editorRoot,
  editorWrapper,
  nestedListItem,
  toolbarRoot,
  readOnlyToolbarRoot,
  toolbarModeSwitch,
  toolbarToggleItem,
  toolbarGroupOfGroups,
  toolbarToggleSingleGroup,
  toolbarButton,
  activeToolbarButton,
  toolbarNodeKindSelectContainer,
  toolbarButtonDropdownContainer,
  toolbarCodeBlockLanguageSelectContent,
  selectContainer,
  selectItem,
  toolbarNodeKindSelectTrigger,
  toolbarButtonSelectTrigger,
  selectTrigger,
  toolbarCodeBlockLanguageSelectTrigger,
  toolbarNodeKindSelectItem,
  toolbarNodeKindSelectDropdownArrow,
  selectDropdownArrow,
  contentEditable,
  sandpackWrapper,
  frontmatterWrapper,
  frontmatterToggleButton,
  propertyPanelTitle,
  propertyEditorTable,
  iconButton,
  readOnlyColumnCell,
  propertyEditorLabelCell,
  buttonsFooter,
  propertyEditorInput,
  primaryButton,
  secondaryButton,
  smallButton,
  dialogForm,
  linkDialogEditForm,
  linkDialogInputContainer,
  linkDialogInputWrapper,
  linkDialogInput,
  dialogInput,
  linkDialogAnchor,
  linkDialogPopoverContent,
  tableColumnEditorPopoverContent,
  dialogContent,
  largeDialogContent,
  dialogTitle,
  dialogCloseButton,
  popoverContent,
  popoverArrow,
  linkDialogPreviewAnchor,
  tooltipTrigger,
  tooltipContent,
  actionButton,
  primaryActionButton,
  tableEditor,
  toolCell,
  tableColumnEditorTrigger,
  tableRowEditorTrigger,
  addRowButton,
  addColumnButton,
  tableColumnEditorToolbar,
  toggleGroupRoot,
  tableToolsColumn,
  leftAlignedCell,
  rightAlignedCell,
  centeredCell,
  dialogOverlay,
  overlayShow,
  contentShow,
  focusedImage,
  imageWrapper,
  editImageButton,
  inlineEditor,
  blockEditor,
  nestedEditor,
  genericComponentName,
  diffSourceToggle,
  selectWithLabel,
  toolbarTitleMode,
  imageControlWrapperResizing,
  imageResizer,
  imageResizerN,
  imageResizerNe,
  imageResizerE,
  imageResizerSe,
  imageResizerS,
  imageResizerSw,
  imageResizerW,
  imageResizerNw,
  placeholder,
  rootContentEditableWrapper,
  downshiftContainer,
  downshiftInputWrapper,
  downshiftInput,
  downshiftAutocompleteContainer,
  textInput,
  multiFieldForm,
  formField,
  markdownParseError,
  popupContainer
};
export {
  actionButton,
  activeToolbarButton,
  addColumnButton,
  addRowButton,
  blockEditor,
  buttonsFooter,
  centeredCell,
  contentEditable,
  contentShow,
  styles as default,
  dialogCloseButton,
  dialogContent,
  dialogForm,
  dialogInput,
  dialogOverlay,
  dialogTitle,
  diffSourceToggle,
  downshiftAutocompleteContainer,
  downshiftContainer,
  downshiftInput,
  downshiftInputWrapper,
  editImageButton,
  editorRoot,
  editorWrapper,
  focusedImage,
  formField,
  frontmatterToggleButton,
  frontmatterWrapper,
  genericComponentName,
  iconButton,
  imageControlWrapperResizing,
  imageResizer,
  imageResizerE,
  imageResizerN,
  imageResizerNe,
  imageResizerNw,
  imageResizerS,
  imageResizerSe,
  imageResizerSw,
  imageResizerW,
  imageWrapper,
  inlineEditor,
  largeDialogContent,
  leftAlignedCell,
  linkDialogAnchor,
  linkDialogEditForm,
  linkDialogInput,
  linkDialogInputContainer,
  linkDialogInputWrapper,
  linkDialogPopoverContent,
  linkDialogPreviewAnchor,
  markdownParseError,
  multiFieldForm,
  nestedEditor,
  nestedListItem,
  overlayShow,
  placeholder,
  popoverArrow,
  popoverContent,
  popupContainer,
  primaryActionButton,
  primaryButton,
  propertyEditorInput,
  propertyEditorLabelCell,
  propertyEditorTable,
  propertyPanelTitle,
  readOnlyColumnCell,
  readOnlyToolbarRoot,
  rightAlignedCell,
  rootContentEditableWrapper,
  sandpackWrapper,
  secondaryButton,
  selectContainer,
  selectDropdownArrow,
  selectItem,
  selectTrigger,
  selectWithLabel,
  smallButton,
  tableColumnEditorPopoverContent,
  tableColumnEditorToolbar,
  tableColumnEditorTrigger,
  tableEditor,
  tableRowEditorTrigger,
  tableToolsColumn,
  textInput,
  toggleGroupRoot,
  toolCell,
  toolbarButton,
  toolbarButtonDropdownContainer,
  toolbarButtonSelectTrigger,
  toolbarCodeBlockLanguageSelectContent,
  toolbarCodeBlockLanguageSelectTrigger,
  toolbarGroupOfGroups,
  toolbarModeSwitch,
  toolbarNodeKindSelectContainer,
  toolbarNodeKindSelectDropdownArrow,
  toolbarNodeKindSelectItem,
  toolbarNodeKindSelectTrigger,
  toolbarRoot,
  toolbarTitleMode,
  toolbarToggleItem,
  toolbarToggleSingleGroup,
  tooltipContent,
  tooltipTrigger
};
