// @mdxeditor/editor/dist/index.js
import "./styles/globals.css.js";
import { MDXEditor } from "./MDXEditor.js";
import { MarkdownParseError, UnrecognizedMarkdownConstructError, importMarkdownToLexical, importMdastTreeToLexical } from "./importMarkdownToLexical.js";
import { exportLexicalTreeToMdast, exportMarkdownFromLexical } from "./exportMarkdownFromLexical.js";
import { NESTED_EDITOR_UPDATED_COMMAND, corePlugin, corePluginHooks, coreSystem } from "./plugins/core/index.js";
import { headingsPlugin, headingsPluginHooks, headingsSystem } from "./plugins/headings/index.js";
import { thematicBreakPlugin, thematicBreakPluginHooks, thematicBreakSystem } from "./plugins/thematic-break/index.js";
import { listsPlugin, listsPluginHooks, listsSystem } from "./plugins/lists/index.js";
import { tablePlugin, tablePluginHooks, tableSystem } from "./plugins/table/index.js";
import { linkPlugin } from "./plugins/link/index.js";
import { INSERT_IMAGE_COMMAND, imagePlugin, imagePluginHooks, imageSystem } from "./plugins/image/index.js";
import { frontmatterPlugin, frontmatterPluginHooks, frontmatterSystem } from "./plugins/frontmatter/index.js";
import { quotePlugin, quotePluginHooks } from "./plugins/quote/index.js";
import { isMdastJsxNode, jsxPlugin, jsxPluginHooks, jsxSystem } from "./plugins/jsx/index.js";
import { GenericJsxEditor } from "./jsx-editors/GenericJsxEditor.js";
import { sandpackPlugin, sandpackPluginHooks, sandpackSystem } from "./plugins/sandpack/index.js";
import { codeMirrorHooks, codeMirrorPlugin, codeMirrorSystem } from "./plugins/codemirror/index.js";
import { CodeMirrorEditor } from "./plugins/codemirror/CodeMirrorEditor.js";
import { codeBlockPlugin, codeBlockPluginHooks, codeBlockSystem } from "./plugins/codeblock/index.js";
import { directivesPlugin, directivesPluginHooks, directivesSystem } from "./plugins/directives/index.js";
import { ADMONITION_TYPES, AdmonitionDirectiveDescriptor } from "./directive-editors/AdmonitionDirectiveDescriptor.js";
import { GenericDirectiveEditor } from "./directive-editors/GenericDirectiveEditor.js";
import { linkDialogPlugin, linkDialogPluginHooks } from "./plugins/link-dialog/index.js";
import { toolbarPlugin, toolbarPluginHooks, toolbarSystem } from "./plugins/toolbar/index.js";
import { diffSourcePlugin, diffSourcePluginHooks, diffSourceSystem } from "./plugins/diff-source/index.js";
import { markdownShortcutPlugin } from "./plugins/markdown-shortcut/index.js";
import { BlockTypeSelect } from "./plugins/toolbar/components/BlockTypeSelect.js";
import { BoldItalicUnderlineToggles } from "./plugins/toolbar/components/BoldItalicUnderlineToggles.js";
import { ChangeAdmonitionType } from "./plugins/toolbar/components/ChangeAdmonitionType.js";
import { ChangeCodeMirrorLanguage } from "./plugins/toolbar/components/ChangeCodeMirrorLanguage.js";
import { CodeToggle } from "./plugins/toolbar/components/CodeToggle.js";
import { CreateLink } from "./plugins/toolbar/components/CreateLink.js";
import { DiffSourceToggleWrapper } from "./plugins/toolbar/components/DiffSourceToggleWrapper.js";
import { InsertAdmonition } from "./plugins/toolbar/components/InsertAdmonition.js";
import { InsertCodeBlock } from "./plugins/toolbar/components/InsertCodeBlock.js";
import { InsertFrontmatter } from "./plugins/toolbar/components/InsertFrontmatter.js";
import { InsertImage } from "./plugins/toolbar/components/InsertImage.js";
import { InsertSandpack } from "./plugins/toolbar/components/InsertSandpack.js";
import { InsertTable } from "./plugins/toolbar/components/InsertTable.js";
import { InsertThematicBreak } from "./plugins/toolbar/components/InsertThematicBreak.js";
import { ListsToggle } from "./plugins/toolbar/components/ListsToggle.js";
import { ShowSandpackInfo } from "./plugins/toolbar/components/ShowSandpackInfo.js";
import { UndoRedo } from "./plugins/toolbar/components/UndoRedo.js";
import { KitchenSinkToolbar } from "./plugins/toolbar/components/KitchenSinkToolbar.js";
import { Button, ButtonOrDropdownButton, ButtonWithTooltip, ConditionalContents, MultipleChoiceToggleGroup, Root, Separator, SingleChoiceToggleGroup, SingleToggleGroup, ToggleSingleGroupWithItem, ToolbarToggleItem } from "./plugins/toolbar/primitives/toolbar.js";
import { DialogButton } from "./plugins/toolbar/primitives/DialogButton.js";
import { TooltipWrap } from "./plugins/toolbar/primitives/TooltipWrap.js";
import { Select, SelectButtonTrigger, SelectContent, SelectItem, SelectTrigger } from "./plugins/toolbar/primitives/select.js";
import { NestedEditorsContext, NestedLexicalEditor, useLexicalNodeRemove, useMdastNodeUpdater, useNestedEditorContext } from "./plugins/core/NestedLexicalEditor.js";
import { PropertyPopover } from "./plugins/core/PropertyPopover.js";
import { CAN_USE_DOM, IS_APPLE, controlOrMeta } from "./utils/detectMac.js";
import { always, call, compose, curry1to0, curry2to1, joinProc, noop, prop, tap, thrush } from "./utils/fp.js";
import { isPartOftheEditorUI } from "./utils/isPartOftheEditorUI.js";
import { fromWithinEditorRead, getSelectedNode, getSelectionRectangle, getStateAsMarkdown } from "./utils/lexicalHelpers.js";
import { makeHslTransparent } from "./utils/makeHslTransparent.js";
import { uuidv4 } from "./utils/uuid4.js";
import { voidEmitter } from "./utils/voidEmitter.js";
import { WHITESPACE_MARKER, replaceWhitespace } from "./utils/whitespaceConversion.js";
import { $createGenericHTMLNode, $isGenericHTMLNode, GenericHTMLNode } from "./plugins/core/GenericHTMLNode.js";
import { $createImageNode, $isImageNode, ImageNode } from "./plugins/image/ImageNode.js";
import { useCodeBlockEditorContext } from "./plugins/codeblock/CodeBlockNode.js";
import { RefCount, defaultComparator, realm } from "./gurx/realm.js";
import { RealmPluginInitializer, RequirePlugin, realmFactoryToComponent, realmPlugin, sysHooks, useHasPlugin } from "./gurx/react.js";
import { getRealmFactory, realmFactory, system } from "./gurx/realmFactory.js";
export {
  $createGenericHTMLNode,
  $createImageNode,
  $isGenericHTMLNode,
  $isImageNode,
  ADMONITION_TYPES,
  AdmonitionDirectiveDescriptor,
  BlockTypeSelect,
  BoldItalicUnderlineToggles,
  Button,
  ButtonOrDropdownButton,
  ButtonWithTooltip,
  CAN_USE_DOM,
  ChangeAdmonitionType,
  ChangeCodeMirrorLanguage,
  CodeMirrorEditor,
  CodeToggle,
  ConditionalContents,
  CreateLink,
  DialogButton,
  DiffSourceToggleWrapper,
  GenericDirectiveEditor,
  GenericHTMLNode,
  GenericJsxEditor,
  INSERT_IMAGE_COMMAND,
  IS_APPLE,
  ImageNode,
  InsertAdmonition,
  InsertCodeBlock,
  InsertFrontmatter,
  InsertImage,
  InsertSandpack,
  InsertTable,
  InsertThematicBreak,
  KitchenSinkToolbar,
  ListsToggle,
  MDXEditor,
  MarkdownParseError,
  MultipleChoiceToggleGroup,
  NESTED_EDITOR_UPDATED_COMMAND,
  NestedEditorsContext,
  NestedLexicalEditor,
  PropertyPopover,
  RealmPluginInitializer,
  RefCount,
  RequirePlugin,
  Root,
  Select,
  SelectButtonTrigger,
  SelectContent,
  SelectItem,
  SelectTrigger,
  Separator,
  ShowSandpackInfo,
  SingleChoiceToggleGroup,
  SingleToggleGroup,
  ToggleSingleGroupWithItem,
  ToolbarToggleItem,
  TooltipWrap,
  UndoRedo,
  UnrecognizedMarkdownConstructError,
  WHITESPACE_MARKER,
  always,
  call,
  codeBlockPlugin,
  codeBlockPluginHooks,
  codeBlockSystem,
  codeMirrorHooks,
  codeMirrorPlugin,
  codeMirrorSystem,
  compose,
  controlOrMeta,
  corePlugin,
  corePluginHooks,
  coreSystem,
  curry1to0,
  curry2to1,
  defaultComparator,
  diffSourcePlugin,
  diffSourcePluginHooks,
  diffSourceSystem,
  directivesPlugin,
  directivesPluginHooks,
  directivesSystem,
  exportLexicalTreeToMdast,
  exportMarkdownFromLexical,
  fromWithinEditorRead,
  frontmatterPlugin,
  frontmatterPluginHooks,
  frontmatterSystem,
  getRealmFactory,
  getSelectedNode,
  getSelectionRectangle,
  getStateAsMarkdown,
  headingsPlugin,
  headingsPluginHooks,
  headingsSystem,
  imagePlugin,
  imagePluginHooks,
  imageSystem,
  importMarkdownToLexical,
  importMdastTreeToLexical,
  isMdastJsxNode,
  isPartOftheEditorUI,
  joinProc,
  jsxPlugin,
  jsxPluginHooks,
  jsxSystem,
  linkDialogPlugin,
  linkDialogPluginHooks,
  linkPlugin,
  listsPlugin,
  listsPluginHooks,
  listsSystem,
  makeHslTransparent,
  markdownShortcutPlugin,
  noop,
  prop,
  quotePlugin,
  quotePluginHooks,
  realm,
  realmFactory,
  realmFactoryToComponent,
  realmPlugin,
  replaceWhitespace,
  sandpackPlugin,
  sandpackPluginHooks,
  sandpackSystem,
  sysHooks,
  system,
  tablePlugin,
  tablePluginHooks,
  tableSystem,
  tap,
  thematicBreakPlugin,
  thematicBreakPluginHooks,
  thematicBreakSystem,
  thrush,
  toolbarPlugin,
  toolbarPluginHooks,
  toolbarSystem,
  useCodeBlockEditorContext,
  useHasPlugin,
  useLexicalNodeRemove,
  useMdastNodeUpdater,
  useNestedEditorContext,
  uuidv4,
  voidEmitter
};
